<a href="https://colab.research.google.com/github/flora0110/sentence_prediction/blob/main/Breeze_example.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [ ]:
import json

with open('/content/drive/MyDrive/senior/LAB/LawCaseRetreiver/sentence_predict/new_dataset/train_data.json', 'r', encoding='utf-8') as f:
    train_data = json.load(f)

with open('/content/drive/MyDrive/senior/LAB/LawCaseRetreiver/sentence_predict/new_dataset/test_data.json', 'r', encoding='utf-8') as f:
    test_data = json.load(f)

In [ ]:
def extract_key_contents(data):
    facts = []
    sentences = []
    for item in data:
        facts.append(item.get('fact'))
        sentences.append(item.get('sentance'))
    return facts, sentences

In [ ]:
train_facts, train_sentences = extract_key_contents(train_data)
test_facts, test_sentences = extract_key_contents(test_data)

# breeze

In [ ]:
import requests
import json

url = "https://nlplab-llm.nlpnchu.org/generate"

headers = {
  'Content-Type': 'application/json',
  'accept': 'application/json'
}

def get_breeze(input_content):
    payload_input=f"<s>You are a helpful AI assistant built by MediaTek Research. The user you are helping speaks Traditional Chinese and comes from Taiwan. [INST] {input_content} [/INST]"
    payload = json.dumps({
    "inputs": payload_input,
    "parameters": {
        "do_sample": True,
        "temperature": 0.01,
        "top_p": 0.95,
        "max_new_tokens": 1000,
    }
    })
    headers = {
    'Content-Type': 'application/json',
    'accept': 'application/json'
    }
    try:
        response = requests.request("POST", url, headers=headers, data=payload)
        j_result=json.loads(response.text)
        if "generated_text" in j_result.keys():
            return j_result['generated_text']
        else:
            return j_result
    except Exception as e:
        print(e)

In [ ]:
print(get_breeze("台灣跟中國的關係"))

 台灣和中國的關係是一個複雜且敏感的議題。目前，台灣是一個由中華民國（Republic of China）所治理的民主國家，而中華人民共和國（People's Republic of China）則主張對台灣擁有主權。自1949年中華人民共和國成立以來，兩岸一直處於分治狀態，但未正式宣告分分。

兩岸的關係主要可以分為以下幾個方面：

1. 政治：中華民國在1949年失去大陸領土後，政府遷至台灣，目前由民選總統領導。中華人民共和國則主張對台灣擁有主權，但由於兩岸政治制度和意識形態不同，導致兩岸官方交流受限。
2. 經濟：兩岸經貿關係密切，主要透過「海峽兩岸經濟合作框架協議」（ECFA）進行。然而，由於政治因素，台灣需要遵循美國的「臺灣關係法」和「六項保證」，限制與中國的經貿合作。
3. 軍事：中國一直以來主張統一台灣，並持續加強軍事力量。台灣則維持足夠的自衛能力，以應對任何可能的威脅。
4. 外交：由於中國的外交壓力，台灣目前只有14個正式外交關係的國家。然而，台灣與許多國家保持良好的非正式關係，並積極參與國際組織。
5. 社會文化：兩岸人民文化上相似，但由於政治因素，交流受到限制。近年來，兩岸民間交流逐漸增加，但官方交流仍受限。

台灣的未來地位仍然是一個未解的問題，可能的結果包括統一、獨立或維持現狀。這個議題在兩岸和國際社會都具有高度敏感性，且影響層面廣泛。


In [ ]:
for i, fact in enumerate(train_facts):
  if(len(fact)<3000):
    print(f"{i}: {len(fact)}")

2: 2608
17: 2766
29: 1192
42: 2585
53: 1867
78: 1525
94: 2361
106: 2013
127: 2565
188: 1259
213: 2919
215: 2691
245: 1777
252: 1559
253: 328
270: 2066
276: 2898
316: 2382
320: 280
324: 2996
328: 1772
335: 2475
395: 1838
398: 2755
409: 1518
416: 2859
420: 2499
425: 2004
426: 1350
432: 2344
438: 1930
460: 1292
490: 2315
496: 1331


# QA

## promp 1
偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境「犯罪情境」。

In [ ]:
query = "偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境"+train_facts[0]+"。 "

In [ ]:
query

'偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境一、曹鳳珠與林寶櫳前為夫妻，因曹鳳珠認為林寶櫳與曹丞君間有不正常的交往關係，2人因此感情生變，遂於民國（下同）99年3月1日在福建連江地方法院調解離婚。詎曹鳳珠為取得保護及教養子女之權利，竟基於行使偽造私文書之犯意，先於某不詳時間、地點，偽造曹丞君簽名及日期94年5月10日之切結書（內容為本人曹丞君因與有婦之夫林寶櫳不正常的密切通聯，被林寶櫳老婆查獲，進而告之本人老公陳葦丞，為求兩家息事寧人，本人在媽媽及姐姐見證下，於姐姐家簽下切結書給林寶櫳老婆曹鳳珠，並保證今後不再與林寶櫳來往，若再被抓到願依妨害家庭處理，此致曹鳳珠）1份，再於99年5月3日，向福建連江地方法院具狀提出而行使之，希冀以此方式取得對子女保護及教養之權利，而足生損害於曹丞君。嗣因林寶櫳於99年5月5日在福建連江地方法院閱卷時，發現上開切結書而向曹丞君詢問，經曹丞君否認後，始查悉上情。二、曹鳳珠因認定係曹丞君介入其與林寶櫳間之婚姻，導致2人離婚，而對曹丞君心懷怨懟，竟基於毀損他人名譽之犯意，先後分別於99年11月5日凌晨4時43分許、100年1月10日上午10時2分許，在其位於連江縣南竿鄉馬祖村85號住處，以SKYPE網路帳號tina900928號，經由網路家庭國際資訊股份有限公司(即PChomeOnline)簡訊系統及米瑟奇網路簡訊平台散佈內容各為「號外！議會之光，春光外洩，曹妓員於11月5日凌晨二點，在神農山莊的休旅車上，與神農山莊王經理幽會長達兩個半小時......。」（下稱簡訊1）、「議員之恥！馬祖之惡！坐領高薪不夠用，議員兼作雞員，曹妓員1月8日深夜三點直奔神農山莊作應召，服務的對象是林姓警員，警紀敗壞蕩然無存；乾爹乾哥變成表哥表弟，破壞別人幸福家庭妻離子散，十惡不赦！罪大惡極！！正義之士群起躂閥！！」（下稱簡訊2）等涉及曹丞君私德而不具公共利益關聯性之事，足以詆毀曹丞君名譽之簡訊。嗣曹丞君陸續接獲他人來電告知收到上開簡訊，始知悉上情。三、案經曹丞君分別

In [ ]:
print(get_breeze(query))

 被告曹鳳珠因行使偽造私文書及誹謗罪，構成犯罪。

一、行使偽造私文書部分：
(一)被告曹鳳珠偽造曹丞君之簽名，並製作切結書，於99年5月3日向福建連江地方法院提出。
(二)被告曹鳳珠偽造之切結書內容為曹丞君因與有婦之夫林寶櫳不正常的交往，被林寶


In [ ]:
query = "偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境"+train_facts[1]+"。 "

In [ ]:
query

'偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境一、利國智與利昀蕙、利芷瑛、利玉秋為兄妹關係。緣利國智等人的父親利錦南於民國104年8月7日死亡前，已將他所有公同及共同共有坐落屏東縣內埔鄉○○段000○000○00000地號土地及同段127建號建物之所有權狀，均交由利玉秋保管。利國智明知前揭土地、建物所有權狀並未遺失，然於利錦南死亡後，利國智為就上開不動產辦理繼承登記，竟基於使公務員登載不實之犯意，於104年8月30日書立「茲就被繼承人利錦南所有（詳後列標示）之所有權狀，因於被繼承人死亡時並未有所交待，且經繼承人多方查覓未果後，已確定遺失屬實」等內容不實之切結書，以前揭所有權狀遺失為由，委由不知情代書曾廷蓉持上開切結書及其他之繼承系統表、戶籍謄本、遺產稅繳（免）納證明等文件，於同年9月1日前往屏東縣潮州地政事務所申請就上開不動產辦理繼承登記，致使不知情之該管公務員，將此不實事項以「編列」代替「登載」方式，將被告之土地登記事項登載於職務上所掌土地登記之電磁紀錄（以公文書論）上（自屬完成登載行為），已足以生損害於利昀蕙、利芷瑛、利玉秋（影響其特留分之權益）及地政機關對於管理不動產之正確性。二、案經利昀蕙、利芷瑛、利玉秋訴由臺灣屏東地方法院檢察署檢察官偵查起訴。理由壹、程序方面：按被告以外之人於審判外之言詞或書面陳述，屬傳聞證據，原則上不得作為證據；惟按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本判決所引用屬於傳聞證據之部分，均已依法踐行調查證據程序，且檢察官、被告於本院審理時，均明示同意有證據能力（見本院卷第41頁），基於尊重當事人對於傳聞證據之處分權，及證據資料愈豐富愈有助於真實發現之理念，本院審酌該等證據作成時情況，並無違法取證之瑕疵，且無顯不可信之情形，以之作為證據應屬適

In [ ]:
print(get_breeze(query))

 被告利國智（下稱被告）因利用不知情的代書曾廷蓉持不實內容的切結書前往潮州地政事務所申請就上開不動產辦理繼承登記，使地政機關承辦人員登載不實，已影響告訴人利昀蕙、利芷瑛、利玉秋（影響其特留分之權益），故被告應成立刑法第214條使公務員登載不實罪。


In [ ]:
query = "偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境"+train_facts[2]+"。 "

In [ ]:
query

'偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境一、陳慶松及其妻左鳳竹均明知陳建妹即陳慶松之母於民國103年11月5日係將苗栗縣○○鄉○○○段000○0地號土地（下稱本案土地）贈與予陳慶松，而非買賣，竟共同基於使公務員登載不實之犯意聯絡，由陳慶松交由左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗縣苗栗地政事務所（下稱苗栗地政事務所）辦理本案土地買賣登記，使不知情之承辦公務員為形式審查後，將「登記原因：買賣」之不實事項登載於職務上所掌之土地登記謄本及地籍異動索引等公文書上，足以生損害於陳建妹及地政機關對於不動產登記之正確管理。二、案經陳建妹訴由臺灣苗栗地方檢察署檢察官偵查起訴。理由壹、證據能力部分一、被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據；被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本案所引被告以外之人於審判外之供述，經被告陳慶松、左鳳竹及辯護人表示同意作為證據（見本院卷第77頁），本院審酌該等陳述無證明力明顯過低之情事，且依製作時之情況，尚無違法不當之瑕疵，認作為證據應屬適當，是得為證據。二、被告等所為不利於己之供述，無出於強暴、脅迫、利誘、詐欺、疲勞訊問、違法羈押或其他不正之方法，亦非違反法定障礙事由經過期間不得訊問或告知義務規定而為，依刑事訴訟法第156條第1項、第158條之2規定，應有證據能力。三、本案所引其餘非供述證據，無違反法定程序而取得之情形，依刑事訴訟法第158條之4反面規定，得為證據。貳、認定犯罪事實所憑證據及其理由訊據被告等固坦承有辦理本案土地買賣登記之事實，惟矢口否認有何使公務員登載不實之犯行，均辯稱：係被告陳慶松以新臺幣（下同）100萬元向告訴人陳建妹購買本案土地云云（見本

In [ ]:
print(get_breeze(query))

 被告陳慶松及其妻左鳳竹均明知本案土地係被告陳慶松之母陳建妹贈與予陳慶松，而非買賣，竟共同基於使公務員登載不實之犯意聯絡，由陳慶松交由左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗縣苗栗地政事務所辦理本案土地買賣登記，使


#### chatgpt ans
一、需為文書：被告人在此案中涉及的文書包括土地登記申請書、土地建築物改良物買賣所有權移轉契約書等文件，以及土地登記謄本、地籍異動索引等公文書。

二、行為人有偽造或變造的行為：被告人陳慶松及其妻左鳳竹明知陳建妹贈與土地给陳慶松，而非買賣，卻向地政機關申報為買賣，將不實事項登載於土地登記謄本等公文書中，以使公務員登載不實之行為。

三、足生損害於公眾或他人：被告人的行為導致土地所有權的登記不正確，损害了陳建妹的权益，并且对地政機關的不動產登記管理造成了損害，因此構成對公眾和他人的損害。

總之，被告人的行為構成偽造文書犯罪的要件，因其明知事實真相卻故意使公務員登載不實，並對他人和公眾造成損害。

In [ ]:
query = "偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境"+train_facts[3]+"。 "

In [ ]:
query

'偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境一、廖麗卿前因偽造有價證券案件，經臺灣高等法院以87年度上訴字第4352號判處有期徒刑2年，並經最高法院以88年度台上字第3995號駁回上訴確定；又因詐欺及偽造文書案件，經臺灣高等法院以88年度上訴字第3284號分別判處有期徒刑1年、3月確定；復因違反動產擔保交易法案件，經臺灣臺北地方法院以87年度店簡字第219號判處有期徒刑3月確定；再因偽造有價證券及偽造文書案件，經臺灣士林地方法院以88年度訴字第733號分別判處有期徒刑4年、8月確定；另因偽造文書案件，經本院以91年度訴字第2628號判處有期徒刑10月確定。上開案件，嗣經本院以92年度聲字第1983號裁定，定其應執行之刑為有期徒刑8年8確，再經本院以98年度聲字第4089號裁定，依中華民國九十六年罪犯減刑條例就應減刑各罪減刑後所處之刑，與不應減刑之罪所處之刑，合併定其應執行有期徒刑7年3月確定，並與其於89年間因竊盜案件，經本院以89年度易字第1253號判處有期徒刑5月，嗣經減刑為有期徒刑2月15日之案件接續執行，於民國96年4月18日縮短刑期假釋出監，假釋期間付保護管束，於96年6月27日保護管束期滿，未經撤銷，視為執行完畢。詎仍不知悔改，於97年初農曆年前，藉由介紹工作而取得鍾賢珍之國民身分證及全民健康保險卡之機會，將鍾賢珍之國民身分證及全民健康保險卡影印留存後，竟與真實姓名年籍不詳之某成年人，基於偽造私文書後進而行使及意圖為自己不法所有之詐欺取財犯意聯絡，於97年5月7日，未經鍾賢珍授權或同意，由廖麗卿在其所任職、由林合昌開設之位於臺中市○○路○段200號之電訪公司，佯以鍾賢珍之名義，申辦行動電話門號，並向該電訪公司不知情之職員偽稱：鍾賢珍為其熟識之友人，其可親自收、送件，無庸透過宅配人員配送行動電話門號空白申請書、SIM卡及附贈之手機云云，致使該公司承辦人員陷於錯誤，而將台灣大哥大股份有限公司（下稱臺灣大哥大公司）之門號0000000000號行動電話SIM卡1枚及

In [ ]:
print(get_breeze(query))

 

犯罪事實

廖麗卿於97年初，因被告之犯罪事實欄所載之犯罪事實，冒用鍾賢珍名義，向其任職之電訪公司申辦行動電話門號0000000000號，並偽造上開行動電話門號之申請書。廖麗卿冒用鍾賢珍名義申辦行動電話門號，並偽造上開行動電話門號之申請書，並持以行使，足


In [ ]:
query = "偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境"+train_facts[4]+"。 "

In [ ]:
query

'偽造文書犯罪的構成要件: 一、需為文書，二、行為人有偽造或變造的行為，三、足生損害於公眾或他人。我會給你一篇判決情境，請你告訴我被告人做了哪些事情，因此構成偽造文書的犯罪要件，越詳細越好。回答時必須以繁體中文輸出並且按照格式回答。格式，一、需為文書: ...。二、行為人有偽造或變造的行為: ...。三、足生損害於公眾或他人: ...。犯罪情境一、高偉為國軍退除役官兵輔導委員會臺北勞務中心（下稱北勞中心）之八德工務所主任，王士章則為上富水土保持技師事務所（下稱上富事務所）之負責人。北勞中心於民國98年5月間取得桃園市楊梅區公所「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程」（下稱老坑溪工程）之工程標，由高偉負責工程施工及工程估驗請款事宜，而老坑溪工程之監造標案，則由上富事務所得標，王士章需負責監督工程及審核廠商估驗請款文件。嗣因向北勞中心承包老坑溪主體工程之工懿營造有限公司延宕工程，經北勞中心解除契約，由擔任履約保證之慶展園藝景觀有限公司（下稱慶展公司）接續施作，慶展公司另將木作工程部分，分包予臺灣綠得電創技有限公司。後因慶展公司負責人林玨豐資金短絀欲退出工程施作，是協議由蕭翰翔出資墊付工程款，繼續以慶展公司名義負責進行工程施作，並覓得蔡盛淵、劉偉任參與工程，蔡盛淵並經北勞中心指派為工地主任，老坑溪工程之文書製作、估驗請款業務則由劉偉任負責，蕭翰翔、蔡盛淵、劉偉任、高偉、王士章就該老坑溪工程，均為從事業務之人。二、蕭翰翔明知「木棧道A」、「木棧道B」、「扶手欄杆」等工程細項，迄民國99年3月15日止，並無任何施作進度，但因負責接續施作工程之蕭翰翔資金週轉陷入困境，計畫先行請領第二期工程款以為挹注，而蔡盛淵、劉偉任、高偉受蕭翰翔請託後，雖明知「木棧道A」、「木棧道B」、「扶手欄杆」確無任何施工進度，但認上揭工程細項所需之施工材料，均已向昆晉實業股份有限公司訂貨進場，日後完成施作應屬無虞，是同意配合，而與蕭翰翔共同基於行使業務登載不實文書之犯意聯絡，由劉偉任在蕭翰翔指示下，於99年3月某日、老坑溪工程工地內，在「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日）」、「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程、第二次估驗數量計算表、99/02/01～99/03/15」、「桃園縣楊梅

In [ ]:
print(get_breeze(query))

 被告蕭翰翔、蔡盛淵、劉偉任、高偉、王士章等人所犯之犯罪行為，符合偽造文書犯罪的構成要件如下：

一、需為文書：被告等人在「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日）」、「楊梅鎮


#### chatgpt ans
一、需為文書: 在這個案例中，涉及了多份文書，包括但不限於「桃園縣楊梅鎮公所公共工程施工日誌」、「楊梅鎮老坑溪兩側路廊景觀工程」等工程估驗請款業務文件。

二、行為人有偽造或變造的行為:
被告人涉嫌偽造文書的行為包括：
1. 登載不實施工進度：被告劉偉任於指示下，在工地內的文件上，如「楊梅鎮公所公共工程施工日誌」及「工程估驗計價明細表」等文書上虛偽登載「木棧道A」、「木棧道B」、「扶手欄杆」等施工項目的進度，實際未有進行相應工作。

2. 蓋印使用公章：被告高偉在工程文件上蓋印「國軍退除役官兵輔導委員會臺北勞務中心」、「曾竹生（北勞中心主任）」的印章，以及被告王士章蓋印「上富水土保持技師事務所」、「王士章」的印文，以表示文件的審核和確認。

3. 簽名：被告蔡盛淵在文件上簽名作為審核的一部分，同時被告高偉在文件上的填表人欄簽名。

以上行為均涉嫌偽造文書或變造文書，並以此來取得工程款項。

三、足生損害於公眾或他人: 在這個案例中，透過偽造的文書，被告人成功取得了工程款項，但實際工程並未按文件所述進行，這對桃園市楊梅區公所的資金使用和工程進度造成了實質性的損害。被告的行為嚴重影響了公共工程的正常進展，也損害了公眾利益。

## promp 2
請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境：「犯罪情境」

In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境："+train_facts[0]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境：一、曹鳳珠與林寶櫳前為夫妻，因曹鳳珠認為林寶櫳與曹丞君間有不正常的交往關係，2人因此感情生變，遂於民國（下同）99年3月1日在福建連江地方法院調解離婚。詎曹鳳珠為取得保護及教養子女之權利，竟基於行使偽造私文書之犯意，先於某不詳時間、地點，偽造曹丞君簽名及日期94年5月10日之切結書（內容為本人曹丞君因與有婦之夫林寶櫳不正常的密切通聯，被林寶櫳老婆查獲，進而告之本人老公陳葦丞，為求兩家息事寧人，本人在媽媽及姐姐見證下，於姐姐家簽下切結書給林寶櫳老婆曹鳳珠，並保證今後不再與林寶櫳來往，若再被抓到願依妨害家庭處理，此致曹鳳珠）1份，再於99年5月3日，向福建連江地方法院具狀提出而行使之，希冀以此方式取得對子女保護及教養之權利，而足生損害於曹丞君。嗣因林寶櫳於99年5月5日在福建連江地方法院閱卷時，發現上開切結書而向曹丞君詢問，經曹丞君否認後，始查悉上情。二、曹鳳珠因認定係曹丞君介入其與林寶櫳間之婚姻，導致2人離婚，而對曹丞君心懷怨懟，竟基於毀損他人名譽之犯意，先後分別於99年11月5日凌晨4時43分許、100年1月10日上午10時2分許，在其位於連江縣南竿鄉馬祖村85號住處，以SKYPE網路帳號tina900928號，經由網路家庭國際資訊股份有限公司(即PChomeOnline)簡訊系統及米瑟奇網路簡訊平台散佈內容各為「號外！議會之光，春光外洩，曹妓員於11月5日凌晨二點，在神農山莊的休旅車上，與神農山莊王經理幽會長達兩個半小時......。」（下稱簡訊1）、「議員之恥！馬祖之惡！坐領高薪不夠用，議員兼作雞員，曹妓員1月8日深夜三點直奔神農山莊作應召，服務的對象是林姓警員，警紀敗壞蕩然無存；乾爹乾哥變成表哥表弟，破壞別人幸福家庭妻離子散，十惡不赦！罪大惡極！！正義之士群起躂閥！！」（下稱簡訊2）等涉及曹丞君私德而不具公共利益關聯性之事，足以詆毀曹丞君名譽之簡訊。嗣曹丞君陸續接獲他人來電告知收到上開簡訊，始知悉上情。三、案經曹丞君分別告訴及訴由福建省連江縣警察局移送福建連江地方法院檢察署檢察官偵查起訴。理由一、按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該

In [ ]:
print(get_breeze(query))

 1. 被告曹鳳珠在離婚後，為了取得對子女的監護權，偽造曹丞君的簽名並撰寫內容，企圖影響法院的監護權判決。

2. 被告曹鳳珠在偽造的切結書上偽造曹丞君的簽名，並撰寫內容，企圖影響法院的監護權判決。

3. 被告曹鳳珠在偽造的切結書上偽造曹丞君的簽名，並撰寫內容，


In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境："+train_facts[1]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境：一、利國智與利昀蕙、利芷瑛、利玉秋為兄妹關係。緣利國智等人的父親利錦南於民國104年8月7日死亡前，已將他所有公同及共同共有坐落屏東縣內埔鄉○○段000○000○00000地號土地及同段127建號建物之所有權狀，均交由利玉秋保管。利國智明知前揭土地、建物所有權狀並未遺失，然於利錦南死亡後，利國智為就上開不動產辦理繼承登記，竟基於使公務員登載不實之犯意，於104年8月30日書立「茲就被繼承人利錦南所有（詳後列標示）之所有權狀，因於被繼承人死亡時並未有所交待，且經繼承人多方查覓未果後，已確定遺失屬實」等內容不實之切結書，以前揭所有權狀遺失為由，委由不知情代書曾廷蓉持上開切結書及其他之繼承系統表、戶籍謄本、遺產稅繳（免）納證明等文件，於同年9月1日前往屏東縣潮州地政事務所申請就上開不動產辦理繼承登記，致使不知情之該管公務員，將此不實事項以「編列」代替「登載」方式，將被告之土地登記事項登載於職務上所掌土地登記之電磁紀錄（以公文書論）上（自屬完成登載行為），已足以生損害於利昀蕙、利芷瑛、利玉秋（影響其特留分之權益）及地政機關對於管理不動產之正確性。二、案經利昀蕙、利芷瑛、利玉秋訴由臺灣屏東地方法院檢察署檢察官偵查起訴。理由壹、程序方面：按被告以外之人於審判外之言詞或書面陳述，屬傳聞證據，原則上不得作為證據；惟按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本判決所引用屬於傳聞證據之部分，均已依法踐行調查證據程序，且檢察官、被告於本院審理時，均明示同意有證據能力（見本院卷第41頁），基於尊重當事人對於傳聞證據之處分權，及證據資料愈豐富愈有助於真實發現之理念，本院審酌該等證據作成時情況，並無違法取證之瑕疵，且無顯不可信之情形，以之作為證據應屬適當，自均有證據能力。貳、實體方面：一、認定犯罪事實所憑之證據及理由：(一)訊據被告利國智（下稱被告）坦承有於104年8月30日曾委由代書曾廷蓉至屏東縣潮州地政事務所，以前揭土地及建物所有權狀遺失為由，填載土

In [ ]:
print(get_breeze(query))

被告利國智（利昀蕙、利芷瑛、利玉秋的父）於民國104年8月30日書立「茲就被繼承人利錦南所有座（詳後列標示）之所有權狀，因於被繼承人死亡時並未有所交待，且經繼承人多方查覓未果後，已確定遺失屬實」等內容之切結書，以前揭所有權狀遺失為由，委由不知情代書


In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境："+train_facts[2]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境：一、陳慶松及其妻左鳳竹均明知陳建妹即陳慶松之母於民國103年11月5日係將苗栗縣○○鄉○○○段000○0地號土地（下稱本案土地）贈與予陳慶松，而非買賣，竟共同基於使公務員登載不實之犯意聯絡，由陳慶松交由左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗縣苗栗地政事務所（下稱苗栗地政事務所）辦理本案土地買賣登記，使不知情之承辦公務員為形式審查後，將「登記原因：買賣」之不實事項登載於職務上所掌之土地登記謄本及地籍異動索引等公文書上，足以生損害於陳建妹及地政機關對於不動產登記之正確管理。二、案經陳建妹訴由臺灣苗栗地方檢察署檢察官偵查起訴。理由壹、證據能力部分一、被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據；被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本案所引被告以外之人於審判外之供述，經被告陳慶松、左鳳竹及辯護人表示同意作為證據（見本院卷第77頁），本院審酌該等陳述無證明力明顯過低之情事，且依製作時之情況，尚無違法不當之瑕疵，認作為證據應屬適當，是得為證據。二、被告等所為不利於己之供述，無出於強暴、脅迫、利誘、詐欺、疲勞訊問、違法羈押或其他不正之方法，亦非違反法定障礙事由經過期間不得訊問或告知義務規定而為，依刑事訴訟法第156條第1項、第158條之2規定，應有證據能力。三、本案所引其餘非供述證據，無違反法定程序而取得之情形，依刑事訴訟法第158條之4反面規定，得為證據。貳、認定犯罪事實所憑證據及其理由訊據被告等固坦承有辦理本案土地買賣登記之事實，惟矢口否認有何使公務員登載不實之犯行，均辯稱：係被告陳慶松以新臺幣（下同）100萬元向告訴人陳建妹購買本案土地云云（見本院卷第75頁至第77頁、第208頁、第284頁）。經查：一、被告左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗地政事務所辦理本案土地買賣登記，使不知情之承

In [ ]:
print(get_breeze(query))

 1.被告偽造了什麼文書？
被告偽造了土地登記申請書、土地建築物改良物買賣所有權移轉契約書等文書。
2.被告做了什麼偽造或變造的行為？
被告陳慶松和左鳳竹共同以不實的買賣原因，向苗栗地政事務所辦理本案土地的買賣登記，使不知情的公務員將不實事項登載於職務上所掌的土地登記謄本、地籍異動索引等


#### chatgpt ans
根據提供的犯罪情境，這是一起涉及偽造土地買賣登記的案件。以下是根據你提供的情境回答的問題：

一、被告偽造了什麼文書？
被告偽造了土地買賣登記相關文件，包括土地登記申請書、土地建築物改良物買賣所有權移轉契約書等。

二、被告做了什麼偽造或變造的行為？
被告偽造了買賣的原因，并以不實的方式向地政機關申報土地買賣交易，使公務員在登記土地所有權移轉時被誤導。

三、造成了什麼損害於公眾或他人的後果？
這種偽造行為對公眾產生了損害，因為土地買賣登記的不實資訊可能會影響到地政機關對土地所有權的正確管理，也可能使不知情的承辦公務員做出錯誤的決定。此外，該行為也對陳建妹的權益造成了損害，因為她是土地贈與者，但被登記為買賣交易的一方。

總結來說，被告的偽造行為損害了公眾利益和他人權益，並對土地買賣交易的合法性造成了影響。

In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境："+train_facts[3]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境：一、廖麗卿前因偽造有價證券案件，經臺灣高等法院以87年度上訴字第4352號判處有期徒刑2年，並經最高法院以88年度台上字第3995號駁回上訴確定；又因詐欺及偽造文書案件，經臺灣高等法院以88年度上訴字第3284號分別判處有期徒刑1年、3月確定；復因違反動產擔保交易法案件，經臺灣臺北地方法院以87年度店簡字第219號判處有期徒刑3月確定；再因偽造有價證券及偽造文書案件，經臺灣士林地方法院以88年度訴字第733號分別判處有期徒刑4年、8月確定；另因偽造文書案件，經本院以91年度訴字第2628號判處有期徒刑10月確定。上開案件，嗣經本院以92年度聲字第1983號裁定，定其應執行之刑為有期徒刑8年8確，再經本院以98年度聲字第4089號裁定，依中華民國九十六年罪犯減刑條例就應減刑各罪減刑後所處之刑，與不應減刑之罪所處之刑，合併定其應執行有期徒刑7年3月確定，並與其於89年間因竊盜案件，經本院以89年度易字第1253號判處有期徒刑5月，嗣經減刑為有期徒刑2月15日之案件接續執行，於民國96年4月18日縮短刑期假釋出監，假釋期間付保護管束，於96年6月27日保護管束期滿，未經撤銷，視為執行完畢。詎仍不知悔改，於97年初農曆年前，藉由介紹工作而取得鍾賢珍之國民身分證及全民健康保險卡之機會，將鍾賢珍之國民身分證及全民健康保險卡影印留存後，竟與真實姓名年籍不詳之某成年人，基於偽造私文書後進而行使及意圖為自己不法所有之詐欺取財犯意聯絡，於97年5月7日，未經鍾賢珍授權或同意，由廖麗卿在其所任職、由林合昌開設之位於臺中市○○路○段200號之電訪公司，佯以鍾賢珍之名義，申辦行動電話門號，並向該電訪公司不知情之職員偽稱：鍾賢珍為其熟識之友人，其可親自收、送件，無庸透過宅配人員配送行動電話門號空白申請書、SIM卡及附贈之手機云云，致使該公司承辦人員陷於錯誤，而將台灣大哥大股份有限公司（下稱臺灣大哥大公司）之門號0000000000號行動電話SIM卡1枚及該電訪公司附隨上開門號SIM卡所贈送之手機1支交付予廖麗卿後，另由上開真實姓名年籍不詳之成年人，擅自在台灣大哥大行動通信網路業務服務申請書之申請人簽章欄內，接續偽造「鍾賢珍」之署名共2枚，而偽造完成表彰係由

In [ ]:
print(get_breeze(query))

 1.犯罪事實：被告廖麗卿於97年5月7日，未經鍾賢珍之同意，以鍾賢珍名義申辦行動電話門號0000000000號行動電話，並偽以鍾賢珍為其熟識之友人，其可親自送件，無庸通


In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境："+train_facts[4]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告偽造了什麼文書？二、被告做了什麼偽造或變造的行為？三、造成了什麼損害於公眾或他人的後果？犯罪情境：一、高偉為國軍退除役官兵輔導委員會臺北勞務中心（下稱北勞中心）之八德工務所主任，王士章則為上富水土保持技師事務所（下稱上富事務所）之負責人。北勞中心於民國98年5月間取得桃園市楊梅區公所「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程」（下稱老坑溪工程）之工程標，由高偉負責工程施工及工程估驗請款事宜，而老坑溪工程之監造標案，則由上富事務所得標，王士章需負責監督工程及審核廠商估驗請款文件。嗣因向北勞中心承包老坑溪主體工程之工懿營造有限公司延宕工程，經北勞中心解除契約，由擔任履約保證之慶展園藝景觀有限公司（下稱慶展公司）接續施作，慶展公司另將木作工程部分，分包予臺灣綠得電創技有限公司。後因慶展公司負責人林玨豐資金短絀欲退出工程施作，是協議由蕭翰翔出資墊付工程款，繼續以慶展公司名義負責進行工程施作，並覓得蔡盛淵、劉偉任參與工程，蔡盛淵並經北勞中心指派為工地主任，老坑溪工程之文書製作、估驗請款業務則由劉偉任負責，蕭翰翔、蔡盛淵、劉偉任、高偉、王士章就該老坑溪工程，均為從事業務之人。二、蕭翰翔明知「木棧道A」、「木棧道B」、「扶手欄杆」等工程細項，迄民國99年3月15日止，並無任何施作進度，但因負責接續施作工程之蕭翰翔資金週轉陷入困境，計畫先行請領第二期工程款以為挹注，而蔡盛淵、劉偉任、高偉受蕭翰翔請託後，雖明知「木棧道A」、「木棧道B」、「扶手欄杆」確無任何施工進度，但認上揭工程細項所需之施工材料，均已向昆晉實業股份有限公司訂貨進場，日後完成施作應屬無虞，是同意配合，而與蕭翰翔共同基於行使業務登載不實文書之犯意聯絡，由劉偉任在蕭翰翔指示下，於99年3月某日、老坑溪工程工地內，在「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日）」、「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程、第二次估驗數量計算表、99/02/01～99/03/15」、「桃園縣楊梅鎮公所工程估驗計價明細表、第二次估驗（估驗期間：99/02/01～99/03/15）」等估驗請款業務文書上，虛偽登載「木棧道A」施工長度110公尺、「木棧道B」施工長度120公尺、「扶手欄杆」施工長度200

In [ ]:
print(get_breeze(query))

 1.被告偽造了什麼文書？
2.被告做了什麼偽造或變造的行為？
3.造成了什麼損害於公眾或他人的後果？

答案：
1.被告蕭翰翔、蔡盛淵、劉偉任、高偉、王士章等人偽造了「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌


#### chatgpt ans
根據提供的犯罪情境，以下是對應的三個問題的回答：

一、被告偽造了什麼文書？
被告涉嫌偽造的文書包括「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程」的相關文件，其中包括但不限於工程施工日誌、工程估驗計價明細表等。

二、被告做了什麼偽造或變造的行為？
被告包括蕭翰翔、劉偉任等人在99年3月的某日，就工程施工進度存在不實之情況下，配合蕭翰翔的指示，在工程文件中虛偽登載「木棧道A」、「木棧道B」、「扶手欄杆」等施工進度，並偽造相關的工程施工日誌、估驗計價明細表等文件。

三、造成了什麼損害於公眾或他人的後果？
這些偽造行為導致了桃園市楊梅區公所在對工程施工進度的稽核中受到損害，使其向慶展公司請款並支付了不實的工程款項，進而造成了公款損失。同时，這也嚴重影響了工程施工的品質和進度的正常監督，損害了公眾利益和工程品質的正常保障。

## promp 3
請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為?
三、被告人損害了被害人的哪些利益? 犯罪情境：「犯罪情境」

In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境："+train_facts[0]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境：一、曹鳳珠與林寶櫳前為夫妻，因曹鳳珠認為林寶櫳與曹丞君間有不正常的交往關係，2人因此感情生變，遂於民國（下同）99年3月1日在福建連江地方法院調解離婚。詎曹鳳珠為取得保護及教養子女之權利，竟基於行使偽造私文書之犯意，先於某不詳時間、地點，偽造曹丞君簽名及日期94年5月10日之切結書（內容為本人曹丞君因與有婦之夫林寶櫳不正常的密切通聯，被林寶櫳老婆查獲，進而告之本人老公陳葦丞，為求兩家息事寧人，本人在媽媽及姐姐見證下，於姐姐家簽下切結書給林寶櫳老婆曹鳳珠，並保證今後不再與林寶櫳來往，若再被抓到願依妨害家庭處理，此致曹鳳珠）1份，再於99年5月3日，向福建連江地方法院具狀提出而行使之，希冀以此方式取得對子女保護及教養之權利，而足生損害於曹丞君。嗣因林寶櫳於99年5月5日在福建連江地方法院閱卷時，發現上開切結書而向曹丞君詢問，經曹丞君否認後，始查悉上情。二、曹鳳珠因認定係曹丞君介入其與林寶櫳間之婚姻，導致2人離婚，而對曹丞君心懷怨懟，竟基於毀損他人名譽之犯意，先後分別於99年11月5日凌晨4時43分許、100年1月10日上午10時2分許，在其位於連江縣南竿鄉馬祖村85號住處，以SKYPE網路帳號tina900928號，經由網路家庭國際資訊股份有限公司(即PChomeOnline)簡訊系統及米瑟奇網路簡訊平台散佈內容各為「號外！議會之光，春光外洩，曹妓員於11月5日凌晨二點，在神農山莊的休旅車上，與神農山莊王經理幽會長達兩個半小時......。」（下稱簡訊1）、「議員之恥！馬祖之惡！坐領高薪不夠用，議員兼作雞員，曹妓員1月8日深夜三點直奔神農山莊作應召，服務的對象是林姓警員，警紀敗壞蕩然無存；乾爹乾哥變成表哥表弟，破壞別人幸福家庭妻離子散，十惡不赦！罪大惡極！！正義之士群起躂閥！！」（下稱簡訊2）等涉及曹丞君私德而不具公共利益關聯性之事，足以詆毀曹丞君名譽之簡訊。嗣曹丞君陸續接獲他人來電告知收到上開簡訊，始知悉上情。三、案經曹丞君分別告訴及訴由福建省連江縣警察局移送福建連江地方法院檢察署檢察官偵查起訴。理由一、按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院

In [ ]:
print(get_breeze(query))

 1. 被告曹鳳珠在福建連江縣法院調解離婚後，偽造曹丞君的簽名並填寫切結書，以取得保護及教養子女的權利。
2. 被告曹鳳珠在未經曹丞君同意下，將曹丞君的姓名、照片、地址等個人資料，散佈於網路，以損害曹丞君的名譽。
3. 被告曹鳳珠偽造曹丞君的簽名，並填寫切結


In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境："+train_facts[1]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境：一、利國智與利昀蕙、利芷瑛、利玉秋為兄妹關係。緣利國智等人的父親利錦南於民國104年8月7日死亡前，已將他所有公同及共同共有坐落屏東縣內埔鄉○○段000○000○00000地號土地及同段127建號建物之所有權狀，均交由利玉秋保管。利國智明知前揭土地、建物所有權狀並未遺失，然於利錦南死亡後，利國智為就上開不動產辦理繼承登記，竟基於使公務員登載不實之犯意，於104年8月30日書立「茲就被繼承人利錦南所有（詳後列標示）之所有權狀，因於被繼承人死亡時並未有所交待，且經繼承人多方查覓未果後，已確定遺失屬實」等內容不實之切結書，以前揭所有權狀遺失為由，委由不知情代書曾廷蓉持上開切結書及其他之繼承系統表、戶籍謄本、遺產稅繳（免）納證明等文件，於同年9月1日前往屏東縣潮州地政事務所申請就上開不動產辦理繼承登記，致使不知情之該管公務員，將此不實事項以「編列」代替「登載」方式，將被告之土地登記事項登載於職務上所掌土地登記之電磁紀錄（以公文書論）上（自屬完成登載行為），已足以生損害於利昀蕙、利芷瑛、利玉秋（影響其特留分之權益）及地政機關對於管理不動產之正確性。二、案經利昀蕙、利芷瑛、利玉秋訴由臺灣屏東地方法院檢察署檢察官偵查起訴。理由壹、程序方面：按被告以外之人於審判外之言詞或書面陳述，屬傳聞證據，原則上不得作為證據；惟按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本判決所引用屬於傳聞證據之部分，均已依法踐行調查證據程序，且檢察官、被告於本院審理時，均明示同意有證據能力（見本院卷第41頁），基於尊重當事人對於傳聞證據之處分權，及證據資料愈豐富愈有助於真實發現之理念，本院審酌該等證據作成時情況，並無違法取證之瑕疵，且無顯不可信之情形，以之作為證據應屬適當，自均有證據能力。貳、實體方面：一、認定犯罪事實所憑之證據及理由：(一)訊據被告利國智（下稱被告）坦承有於104年8月30日曾委由代書曾廷蓉至屏東縣潮州地政事務所，以前揭土地及建物所有權狀遺失為由，

In [ ]:
print(get_breeze(query))

被告利國智（下稱被告）利用不知情的代書曾廷蓉，以不實的切結書向潮州地政事務所申請就上開不動產辦理繼承登記，使地政機關承辦人員將不實事項登載於職務上所掌的土地登記之電磁紀錄（以公文書論）上。被告的行為已足生損害於告訴人利昀蕙、利芷瑛、利玉秋（影響其特留分的權益）及地政機關


In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境："+train_facts[2]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境：一、陳慶松及其妻左鳳竹均明知陳建妹即陳慶松之母於民國103年11月5日係將苗栗縣○○鄉○○○段000○0地號土地（下稱本案土地）贈與予陳慶松，而非買賣，竟共同基於使公務員登載不實之犯意聯絡，由陳慶松交由左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗縣苗栗地政事務所（下稱苗栗地政事務所）辦理本案土地買賣登記，使不知情之承辦公務員為形式審查後，將「登記原因：買賣」之不實事項登載於職務上所掌之土地登記謄本及地籍異動索引等公文書上，足以生損害於陳建妹及地政機關對於不動產登記之正確管理。二、案經陳建妹訴由臺灣苗栗地方檢察署檢察官偵查起訴。理由壹、證據能力部分一、被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據；被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本案所引被告以外之人於審判外之供述，經被告陳慶松、左鳳竹及辯護人表示同意作為證據（見本院卷第77頁），本院審酌該等陳述無證明力明顯過低之情事，且依製作時之情況，尚無違法不當之瑕疵，認作為證據應屬適當，是得為證據。二、被告等所為不利於己之供述，無出於強暴、脅迫、利誘、詐欺、疲勞訊問、違法羈押或其他不正之方法，亦非違反法定障礙事由經過期間不得訊問或告知義務規定而為，依刑事訴訟法第156條第1項、第158條之2規定，應有證據能力。三、本案所引其餘非供述證據，無違反法定程序而取得之情形，依刑事訴訟法第158條之4反面規定，得為證據。貳、認定犯罪事實所憑證據及其理由訊據被告等固坦承有辦理本案土地買賣登記之事實，惟矢口否認有何使公務員登載不實之犯行，均辯稱：係被告陳慶松以新臺幣（下同）100萬元向告訴人陳建妹購買本案土地云云（見本院卷第75頁至第77頁、第208頁、第284頁）。經查：一、被告左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗地政事務所辦理本案土地買賣登記，使不知

In [ ]:
print(get_breeze(query))

 1.被告人偽造了哪些文書？
被告人偽造了土地登記申請書、土地建築物改良物買賣所有權移轉契約書等文書。
2.被告人做了哪些偽造或變造的行為？
被告人共同偽造了土地買賣的交易事實，將告訴人贈與的土地以買賣的名義辦理過戶，並使不知情的地政事務所公務員登載不實。
3.被告人損害了被害人的哪些利益


In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境："+train_facts[3]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境：一、廖麗卿前因偽造有價證券案件，經臺灣高等法院以87年度上訴字第4352號判處有期徒刑2年，並經最高法院以88年度台上字第3995號駁回上訴確定；又因詐欺及偽造文書案件，經臺灣高等法院以88年度上訴字第3284號分別判處有期徒刑1年、3月確定；復因違反動產擔保交易法案件，經臺灣臺北地方法院以87年度店簡字第219號判處有期徒刑3月確定；再因偽造有價證券及偽造文書案件，經臺灣士林地方法院以88年度訴字第733號分別判處有期徒刑4年、8月確定；另因偽造文書案件，經本院以91年度訴字第2628號判處有期徒刑10月確定。上開案件，嗣經本院以92年度聲字第1983號裁定，定其應執行之刑為有期徒刑8年8確，再經本院以98年度聲字第4089號裁定，依中華民國九十六年罪犯減刑條例就應減刑各罪減刑後所處之刑，與不應減刑之罪所處之刑，合併定其應執行有期徒刑7年3月確定，並與其於89年間因竊盜案件，經本院以89年度易字第1253號判處有期徒刑5月，嗣經減刑為有期徒刑2月15日之案件接續執行，於民國96年4月18日縮短刑期假釋出監，假釋期間付保護管束，於96年6月27日保護管束期滿，未經撤銷，視為執行完畢。詎仍不知悔改，於97年初農曆年前，藉由介紹工作而取得鍾賢珍之國民身分證及全民健康保險卡之機會，將鍾賢珍之國民身分證及全民健康保險卡影印留存後，竟與真實姓名年籍不詳之某成年人，基於偽造私文書後進而行使及意圖為自己不法所有之詐欺取財犯意聯絡，於97年5月7日，未經鍾賢珍授權或同意，由廖麗卿在其所任職、由林合昌開設之位於臺中市○○路○段200號之電訪公司，佯以鍾賢珍之名義，申辦行動電話門號，並向該電訪公司不知情之職員偽稱：鍾賢珍為其熟識之友人，其可親自收、送件，無庸透過宅配人員配送行動電話門號空白申請書、SIM卡及附贈之手機云云，致使該公司承辦人員陷於錯誤，而將台灣大哥大股份有限公司（下稱臺灣大哥大公司）之門號0000000000號行動電話SIM卡1枚及該電訪公司附隨上開門號SIM卡所贈送之手機1支交付予廖麗卿後，另由上開真實姓名年籍不詳之成年人，擅自在台灣大哥大行動通信網路業務服務申請書之申請人簽章欄內，接續偽造「鍾賢珍」之署名共2枚，而偽造完成表

In [ ]:
print(get_breeze(query))

 1.犯罪事實：被告廖麗卿於97年5月7日，未經鍾賢珍同意，以鍾賢珍名義申辦行動電話門號0000000000號行動電話，並偽造有「鍾賢珍」之署名共2枚，偽造有「鍾賢珍」之私文書，並行使之，以詐術使人將本人或第三人之物交付予被告，使被告得以獲得行動電話SIM


In [ ]:
query = "請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境："+train_facts[4]

In [ ]:
query

'請依照以下犯罪情境回答三個問題：一、被告人偽造了哪些文書? 二、被告人做了哪些偽造或變造的行為? 三、被告人損害了被害人的哪些利益? 犯罪情境：一、高偉為國軍退除役官兵輔導委員會臺北勞務中心（下稱北勞中心）之八德工務所主任，王士章則為上富水土保持技師事務所（下稱上富事務所）之負責人。北勞中心於民國98年5月間取得桃園市楊梅區公所「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程」（下稱老坑溪工程）之工程標，由高偉負責工程施工及工程估驗請款事宜，而老坑溪工程之監造標案，則由上富事務所得標，王士章需負責監督工程及審核廠商估驗請款文件。嗣因向北勞中心承包老坑溪主體工程之工懿營造有限公司延宕工程，經北勞中心解除契約，由擔任履約保證之慶展園藝景觀有限公司（下稱慶展公司）接續施作，慶展公司另將木作工程部分，分包予臺灣綠得電創技有限公司。後因慶展公司負責人林玨豐資金短絀欲退出工程施作，是協議由蕭翰翔出資墊付工程款，繼續以慶展公司名義負責進行工程施作，並覓得蔡盛淵、劉偉任參與工程，蔡盛淵並經北勞中心指派為工地主任，老坑溪工程之文書製作、估驗請款業務則由劉偉任負責，蕭翰翔、蔡盛淵、劉偉任、高偉、王士章就該老坑溪工程，均為從事業務之人。二、蕭翰翔明知「木棧道A」、「木棧道B」、「扶手欄杆」等工程細項，迄民國99年3月15日止，並無任何施作進度，但因負責接續施作工程之蕭翰翔資金週轉陷入困境，計畫先行請領第二期工程款以為挹注，而蔡盛淵、劉偉任、高偉受蕭翰翔請託後，雖明知「木棧道A」、「木棧道B」、「扶手欄杆」確無任何施工進度，但認上揭工程細項所需之施工材料，均已向昆晉實業股份有限公司訂貨進場，日後完成施作應屬無虞，是同意配合，而與蕭翰翔共同基於行使業務登載不實文書之犯意聯絡，由劉偉任在蕭翰翔指示下，於99年3月某日、老坑溪工程工地內，在「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日）」、「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程、第二次估驗數量計算表、99/02/01～99/03/15」、「桃園縣楊梅鎮公所工程估驗計價明細表、第二次估驗（估驗期間：99/02/01～99/03/15）」等估驗請款業務文書上，虛偽登載「木棧道A」施工長度110公尺、「木棧道B」施工長度120公尺、「扶手欄杆」施工長度

In [ ]:
print(get_breeze(query))

 1. 被告蕭翰翔、蔡盛淵、劉偉任、高偉、王士章偽造了哪些文書？
2. 被告蕭翰翔、蔡盛淵、劉偉任、高偉、王士章偽造了「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日


## promp 4
請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境：「犯罪情境」

In [ ]:
query = "請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境："+train_facts[0]

In [ ]:
query

'請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境：一、曹鳳珠與林寶櫳前為夫妻，因曹鳳珠認為林寶櫳與曹丞君間有不正常的交往關係，2人因此感情生變，遂於民國（下同）99年3月1日在福建連江地方法院調解離婚。詎曹鳳珠為取得保護及教養子女之權利，竟基於行使偽造私文書之犯意，先於某不詳時間、地點，偽造曹丞君簽名及日期94年5月10日之切結書（內容為本人曹丞君因與有婦之夫林寶櫳不正常的密切通聯，被林寶櫳老婆查獲，進而告之本人老公陳葦丞，為求兩家息事寧人，本人在媽媽及姐姐見證下，於姐姐家簽下切結書給林寶櫳老婆曹鳳珠，並保證今後不再與林寶櫳來往，若再被抓到願依妨害家庭處理，此致曹鳳珠）1份，再於99年5月3日，向福建連江地方法院具狀提出而行使之，希冀以此方式取得對子女保護及教養之權利，而足生損害於曹丞君。嗣因林寶櫳於99年5月5日在福建連江地方法院閱卷時，發現上開切結書而向曹丞君詢問，經曹丞君否認後，始查悉上情。二、曹鳳珠因認定係曹丞君介入其與林寶櫳間之婚姻，導致2人離婚，而對曹丞君心懷怨懟，竟基於毀損他人名譽之犯意，先後分別於99年11月5日凌晨4時43分許、100年1月10日上午10時2分許，在其位於連江縣南竿鄉馬祖村85號住處，以SKYPE網路帳號tina900928號，經由網路家庭國際資訊股份有限公司(即PChomeOnline)簡訊系統及米瑟奇網路簡訊平台散佈內容各為「號外！議會之光，春光外洩，曹妓員於11月5日凌晨二點，在神農山莊的休旅車上，與神農山莊王經理幽會長達兩個半小時......。」（下稱簡訊1）、「議員之恥！馬祖之惡！坐領高薪不夠用，議員兼作雞員，曹妓員1月8日深夜三點直奔神農山莊作應召，服務的對象是林姓警員，警紀敗壞蕩然無存；乾爹乾哥變成表哥表弟，破壞別人幸福家庭妻離子散，十惡不赦！罪大惡極！！正義之士群起躂閥！！」（下稱簡訊2）等涉及曹丞君私德而不具公共利益關聯性之事，足以詆毀曹丞君名譽之簡訊。嗣曹丞君陸續接獲他人來電告知收到上開簡訊，始知悉上情。三、案經曹丞君分別告訴及訴由福建省連江縣警察局移送福建連江地方法院檢察署檢察官偵查起訴。理由一、按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據。當事人、代理人或辯護人於

In [ ]:
print(get_breeze(query))

 1. 被告曹鳳珠在99年3月1日與曹丞君簽署切結書，內容為「本人曹丞君因與有婦之夫林寶櫳不正常的密切通聯，被林寶櫳老婆查獲，進而告之本人老公陳葦丞，為求兩家息事寧人，本人在媽媽及姐姐見證下，於姐姐家簽下切結書給林寶櫳，並保證今後不再與林寶櫳來往


In [ ]:
query = "請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境："+train_facts[1]

In [ ]:
query

'請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境：一、利國智與利昀蕙、利芷瑛、利玉秋為兄妹關係。緣利國智等人的父親利錦南於民國104年8月7日死亡前，已將他所有公同及共同共有坐落屏東縣內埔鄉○○段000○000○00000地號土地及同段127建號建物之所有權狀，均交由利玉秋保管。利國智明知前揭土地、建物所有權狀並未遺失，然於利錦南死亡後，利國智為就上開不動產辦理繼承登記，竟基於使公務員登載不實之犯意，於104年8月30日書立「茲就被繼承人利錦南所有（詳後列標示）之所有權狀，因於被繼承人死亡時並未有所交待，且經繼承人多方查覓未果後，已確定遺失屬實」等內容不實之切結書，以前揭所有權狀遺失為由，委由不知情代書曾廷蓉持上開切結書及其他之繼承系統表、戶籍謄本、遺產稅繳（免）納證明等文件，於同年9月1日前往屏東縣潮州地政事務所申請就上開不動產辦理繼承登記，致使不知情之該管公務員，將此不實事項以「編列」代替「登載」方式，將被告之土地登記事項登載於職務上所掌土地登記之電磁紀錄（以公文書論）上（自屬完成登載行為），已足以生損害於利昀蕙、利芷瑛、利玉秋（影響其特留分之權益）及地政機關對於管理不動產之正確性。二、案經利昀蕙、利芷瑛、利玉秋訴由臺灣屏東地方法院檢察署檢察官偵查起訴。理由壹、程序方面：按被告以外之人於審判外之言詞或書面陳述，屬傳聞證據，原則上不得作為證據；惟按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本判決所引用屬於傳聞證據之部分，均已依法踐行調查證據程序，且檢察官、被告於本院審理時，均明示同意有證據能力（見本院卷第41頁），基於尊重當事人對於傳聞證據之處分權，及證據資料愈豐富愈有助於真實發現之理念，本院審酌該等證據作成時情況，並無違法取證之瑕疵，且無顯不可信之情形，以之作為證據應屬適當，自均有證據能力。貳、實體方面：一、認定犯罪事實所憑之證據及理由：(一)訊據被告利國智（下稱被告）坦承有於104年8月30日曾委由代書曾廷蓉至屏東縣潮州地政事務所，以前揭土地及建物所有權狀遺失為由，填載土地登記申請書、切結書而申請就上開不動產辦理繼承登記等事實，惟矢口否認有何使公務員

In [ ]:
print(get_breeze(query))

被告利國智（Liu Guozhi）偽造文書的犯罪行為涉及以下事項：

1. 在104年8月30日，被告利國智（Liu Guozhi）向屏東縣潮州地政事務所申請就坐落屏東縣內埔鄉○○段000○000○00000地號土地及同段127建號建物之所有權狀遺失為由，辦理繼承登記。



In [ ]:
query = "請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境："+train_facts[2]

In [ ]:
query

'請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境：一、陳慶松及其妻左鳳竹均明知陳建妹即陳慶松之母於民國103年11月5日係將苗栗縣○○鄉○○○段000○0地號土地（下稱本案土地）贈與予陳慶松，而非買賣，竟共同基於使公務員登載不實之犯意聯絡，由陳慶松交由左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗縣苗栗地政事務所（下稱苗栗地政事務所）辦理本案土地買賣登記，使不知情之承辦公務員為形式審查後，將「登記原因：買賣」之不實事項登載於職務上所掌之土地登記謄本及地籍異動索引等公文書上，足以生損害於陳建妹及地政機關對於不動產登記之正確管理。二、案經陳建妹訴由臺灣苗栗地方檢察署檢察官偵查起訴。理由壹、證據能力部分一、被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據；被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本案所引被告以外之人於審判外之供述，經被告陳慶松、左鳳竹及辯護人表示同意作為證據（見本院卷第77頁），本院審酌該等陳述無證明力明顯過低之情事，且依製作時之情況，尚無違法不當之瑕疵，認作為證據應屬適當，是得為證據。二、被告等所為不利於己之供述，無出於強暴、脅迫、利誘、詐欺、疲勞訊問、違法羈押或其他不正之方法，亦非違反法定障礙事由經過期間不得訊問或告知義務規定而為，依刑事訴訟法第156條第1項、第158條之2規定，應有證據能力。三、本案所引其餘非供述證據，無違反法定程序而取得之情形，依刑事訴訟法第158條之4反面規定，得為證據。貳、認定犯罪事實所憑證據及其理由訊據被告等固坦承有辦理本案土地買賣登記之事實，惟矢口否認有何使公務員登載不實之犯行，均辯稱：係被告陳慶松以新臺幣（下同）100萬元向告訴人陳建妹購買本案土地云云（見本院卷第75頁至第77頁、第208頁、第284頁）。經查：一、被告左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗地政事務所辦理本案土地買賣登記，使不知情之承辦公務員為形式審查後，將「登記原因：買賣」之事項登載於職務上所掌之土地登記謄本及

In [ ]:
print(get_breeze(query))

 1. 被告陳慶松及其妻左鳳竹均偽造了哪些文書？
2. 被告陳慶松及其妻左鳳竹偽造了土地登記申請書、土地建築物改良物買賣所有權移轉契約書等文件。


#### chatgpt ans
根據所提供的犯罪情境，被告人偽造了以下文書：

1. 土地登記申請書：被告人左鳳竹持有土地登記申請書，向苗栗地政事務所辦理本案土地買賣登記。在這份文件中，被告人使公務員登載了不實的事項，即將「登記原因：買賣」之事項登載於公文書上。

2. 土地建築物改良物買賣所有權移轉契約書：同樣地，土地建築物改良物買賣所有權移轉契約書也是被告人

的偽造文件之一。這份契約書被用來支持土地買賣的虛假交易。

這些文件的偽造使得不動產交易的實際情況被歪曲，對於地政機關和其他當事人的權益都造成了損害。

In [ ]:
query = "請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境："+train_facts[3]

In [ ]:
query

'請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境：一、廖麗卿前因偽造有價證券案件，經臺灣高等法院以87年度上訴字第4352號判處有期徒刑2年，並經最高法院以88年度台上字第3995號駁回上訴確定；又因詐欺及偽造文書案件，經臺灣高等法院以88年度上訴字第3284號分別判處有期徒刑1年、3月確定；復因違反動產擔保交易法案件，經臺灣臺北地方法院以87年度店簡字第219號判處有期徒刑3月確定；再因偽造有價證券及偽造文書案件，經臺灣士林地方法院以88年度訴字第733號分別判處有期徒刑4年、8月確定；另因偽造文書案件，經本院以91年度訴字第2628號判處有期徒刑10月確定。上開案件，嗣經本院以92年度聲字第1983號裁定，定其應執行之刑為有期徒刑8年8確，再經本院以98年度聲字第4089號裁定，依中華民國九十六年罪犯減刑條例就應減刑各罪減刑後所處之刑，與不應減刑之罪所處之刑，合併定其應執行有期徒刑7年3月確定，並與其於89年間因竊盜案件，經本院以89年度易字第1253號判處有期徒刑5月，嗣經減刑為有期徒刑2月15日之案件接續執行，於民國96年4月18日縮短刑期假釋出監，假釋期間付保護管束，於96年6月27日保護管束期滿，未經撤銷，視為執行完畢。詎仍不知悔改，於97年初農曆年前，藉由介紹工作而取得鍾賢珍之國民身分證及全民健康保險卡之機會，將鍾賢珍之國民身分證及全民健康保險卡影印留存後，竟與真實姓名年籍不詳之某成年人，基於偽造私文書後進而行使及意圖為自己不法所有之詐欺取財犯意聯絡，於97年5月7日，未經鍾賢珍授權或同意，由廖麗卿在其所任職、由林合昌開設之位於臺中市○○路○段200號之電訪公司，佯以鍾賢珍之名義，申辦行動電話門號，並向該電訪公司不知情之職員偽稱：鍾賢珍為其熟識之友人，其可親自收、送件，無庸透過宅配人員配送行動電話門號空白申請書、SIM卡及附贈之手機云云，致使該公司承辦人員陷於錯誤，而將台灣大哥大股份有限公司（下稱臺灣大哥大公司）之門號0000000000號行動電話SIM卡1枚及該電訪公司附隨上開門號SIM卡所贈送之手機1支交付予廖麗卿後，另由上開真實姓名年籍不詳之成年人，擅自在台灣大哥大行動通信網路業務服務申請書之申請人簽章欄內，接續偽造「鍾賢珍」之署名共2枚，而偽造完成表彰係由「鍾賢珍」申請上開行動電話門號，性質上屬於私文書之申請書1份後，再將該份申請書交

In [ ]:
print(get_breeze(query))

 1.犯罪事實：被告廖麗珍於97年5月7日，未經鍾賢珍之同意或授權，冒用鍾賢珍名義申辦行動電話門號0000000000號行動電話，並偽造有「鍾賢珍」之署名，持以行使之，向其任職之電訪公司承辦人員申領行動電話SIM卡1枚及附贈之手機1支，並偽造臺灣大哥大公司之


In [ ]:
query = "請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境："+train_facts[4]

In [ ]:
query

'請依照以下犯罪情境告訴我被告人偽造了哪些文書? 犯罪情境：一、高偉為國軍退除役官兵輔導委員會臺北勞務中心（下稱北勞中心）之八德工務所主任，王士章則為上富水土保持技師事務所（下稱上富事務所）之負責人。北勞中心於民國98年5月間取得桃園市楊梅區公所「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程」（下稱老坑溪工程）之工程標，由高偉負責工程施工及工程估驗請款事宜，而老坑溪工程之監造標案，則由上富事務所得標，王士章需負責監督工程及審核廠商估驗請款文件。嗣因向北勞中心承包老坑溪主體工程之工懿營造有限公司延宕工程，經北勞中心解除契約，由擔任履約保證之慶展園藝景觀有限公司（下稱慶展公司）接續施作，慶展公司另將木作工程部分，分包予臺灣綠得電創技有限公司。後因慶展公司負責人林玨豐資金短絀欲退出工程施作，是協議由蕭翰翔出資墊付工程款，繼續以慶展公司名義負責進行工程施作，並覓得蔡盛淵、劉偉任參與工程，蔡盛淵並經北勞中心指派為工地主任，老坑溪工程之文書製作、估驗請款業務則由劉偉任負責，蕭翰翔、蔡盛淵、劉偉任、高偉、王士章就該老坑溪工程，均為從事業務之人。二、蕭翰翔明知「木棧道A」、「木棧道B」、「扶手欄杆」等工程細項，迄民國99年3月15日止，並無任何施作進度，但因負責接續施作工程之蕭翰翔資金週轉陷入困境，計畫先行請領第二期工程款以為挹注，而蔡盛淵、劉偉任、高偉受蕭翰翔請託後，雖明知「木棧道A」、「木棧道B」、「扶手欄杆」確無任何施工進度，但認上揭工程細項所需之施工材料，均已向昆晉實業股份有限公司訂貨進場，日後完成施作應屬無虞，是同意配合，而與蕭翰翔共同基於行使業務登載不實文書之犯意聯絡，由劉偉任在蕭翰翔指示下，於99年3月某日、老坑溪工程工地內，在「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日）」、「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程、第二次估驗數量計算表、99/02/01～99/03/15」、「桃園縣楊梅鎮公所工程估驗計價明細表、第二次估驗（估驗期間：99/02/01～99/03/15）」等估驗請款業務文書上，虛偽登載「木棧道A」施工長度110公尺、「木棧道B」施工長度120公尺、「扶手欄杆」施工長度200公尺之不實施工進度，北勞中心派任之工地主任蔡盛淵則於「桃園縣楊梅鎮公所公共工程施

In [ ]:
print(get_breeze(query))

被告蕭翰翔、蔡盛淵、劉偉任、高偉、王士章所犯之犯罪行為，係觸犯刑法第216條、第215條之行使業務上登載不實文書罪。


#### chatgpt ans
根据所提供的犯罪情境，被告人涉嫌偽造的文件如下：

1. **「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程、第二次估驗數量計算表、99/02/01-99/03/15」：** 在这份文件中，被告们虚偽登載了「木棧道A」施工长度110公尺、「木棧道B」施工长度120公尺、「扶手欄杆」施工长度200公尺的不实施工进度。

2. **「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」和「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日）」：** 在这两份工程施工日誌中，被告们虚偽地填写了施工进度，声称已完成了某些工程细项的施工，实际上没有进行施工。

3. **「桃園縣楊梅鎮公所工程估驗計價明細表、第二次估驗（估驗期間：99/02/01～99/03/15）」：** 这份估验计价明细表中可能包含了被告们虚偽登记的工程进度和费用。

以上这些文件在工程进度和费用上存在不实登记，被告人可能通过这些虚偽文件获得了工程款项，构成了行使業務上登載不實文書罪。

## promp 5
請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境：「犯罪情境」

In [ ]:
query = "請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境："+train_facts[0]

In [ ]:
query

'請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境：一、曹鳳珠與林寶櫳前為夫妻，因曹鳳珠認為林寶櫳與曹丞君間有不正常的交往關係，2人因此感情生變，遂於民國（下同）99年3月1日在福建連江地方法院調解離婚。詎曹鳳珠為取得保護及教養子女之權利，竟基於行使偽造私文書之犯意，先於某不詳時間、地點，偽造曹丞君簽名及日期94年5月10日之切結書（內容為本人曹丞君因與有婦之夫林寶櫳不正常的密切通聯，被林寶櫳老婆查獲，進而告之本人老公陳葦丞，為求兩家息事寧人，本人在媽媽及姐姐見證下，於姐姐家簽下切結書給林寶櫳老婆曹鳳珠，並保證今後不再與林寶櫳來往，若再被抓到願依妨害家庭處理，此致曹鳳珠）1份，再於99年5月3日，向福建連江地方法院具狀提出而行使之，希冀以此方式取得對子女保護及教養之權利，而足生損害於曹丞君。嗣因林寶櫳於99年5月5日在福建連江地方法院閱卷時，發現上開切結書而向曹丞君詢問，經曹丞君否認後，始查悉上情。二、曹鳳珠因認定係曹丞君介入其與林寶櫳間之婚姻，導致2人離婚，而對曹丞君心懷怨懟，竟基於毀損他人名譽之犯意，先後分別於99年11月5日凌晨4時43分許、100年1月10日上午10時2分許，在其位於連江縣南竿鄉馬祖村85號住處，以SKYPE網路帳號tina900928號，經由網路家庭國際資訊股份有限公司(即PChomeOnline)簡訊系統及米瑟奇網路簡訊平台散佈內容各為「號外！議會之光，春光外洩，曹妓員於11月5日凌晨二點，在神農山莊的休旅車上，與神農山莊王經理幽會長達兩個半小時......。」（下稱簡訊1）、「議員之恥！馬祖之惡！坐領高薪不夠用，議員兼作雞員，曹妓員1月8日深夜三點直奔神農山莊作應召，服務的對象是林姓警員，警紀敗壞蕩然無存；乾爹乾哥變成表哥表弟，破壞別人幸福家庭妻離子散，十惡不赦！罪大惡極！！正義之士群起躂閥！！」（下稱簡訊2）等涉及曹丞君私德而不具公共利益關聯性之事，足以詆毀曹丞君名譽之簡訊。嗣曹丞君陸續接獲他人來電告知收到上開簡訊，始知悉上情。三、案經曹丞君分別告訴及訴由福建省連江縣警察局移送福建連江地方法院檢察署檢察官偵查起訴。理由一、按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據。當事人、代理人

In [ ]:
print(get_breeze(query))

 

被告曹鳳珠在離婚後，為了取得對子女的監護權，偽造曹丞君的簽名並撰寫內容，以取得曹丞君的切結書。被告在訴訟中提出該偽造的切結書，以取得監護權。被告的行為已侵害曹丞君的權益，因此被告曹鳳珠觸犯行使偽造私文書罪。

被告曹鳳珠在網路上散佈的簡訊內容，已損害曹丞君的名譽


In [ ]:
query = "請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境："+train_facts[1]

In [ ]:
query

'請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境：一、利國智與利昀蕙、利芷瑛、利玉秋為兄妹關係。緣利國智等人的父親利錦南於民國104年8月7日死亡前，已將他所有公同及共同共有坐落屏東縣內埔鄉○○段000○000○00000地號土地及同段127建號建物之所有權狀，均交由利玉秋保管。利國智明知前揭土地、建物所有權狀並未遺失，然於利錦南死亡後，利國智為就上開不動產辦理繼承登記，竟基於使公務員登載不實之犯意，於104年8月30日書立「茲就被繼承人利錦南所有（詳後列標示）之所有權狀，因於被繼承人死亡時並未有所交待，且經繼承人多方查覓未果後，已確定遺失屬實」等內容不實之切結書，以前揭所有權狀遺失為由，委由不知情代書曾廷蓉持上開切結書及其他之繼承系統表、戶籍謄本、遺產稅繳（免）納證明等文件，於同年9月1日前往屏東縣潮州地政事務所申請就上開不動產辦理繼承登記，致使不知情之該管公務員，將此不實事項以「編列」代替「登載」方式，將被告之土地登記事項登載於職務上所掌土地登記之電磁紀錄（以公文書論）上（自屬完成登載行為），已足以生損害於利昀蕙、利芷瑛、利玉秋（影響其特留分之權益）及地政機關對於管理不動產之正確性。二、案經利昀蕙、利芷瑛、利玉秋訴由臺灣屏東地方法院檢察署檢察官偵查起訴。理由壹、程序方面：按被告以外之人於審判外之言詞或書面陳述，屬傳聞證據，原則上不得作為證據；惟按被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本判決所引用屬於傳聞證據之部分，均已依法踐行調查證據程序，且檢察官、被告於本院審理時，均明示同意有證據能力（見本院卷第41頁），基於尊重當事人對於傳聞證據之處分權，及證據資料愈豐富愈有助於真實發現之理念，本院審酌該等證據作成時情況，並無違法取證之瑕疵，且無顯不可信之情形，以之作為證據應屬適當，自均有證據能力。貳、實體方面：一、認定犯罪事實所憑之證據及理由：(一)訊據被告利國智（下稱被告）坦承有於104年8月30日曾委由代書曾廷蓉至屏東縣潮州地政事務所，以前揭土地及建物所有權狀遺失為由，填載土地登記申請書、切結書而申請就上開不動產辦理繼承登記等事實，惟矢口否認有

In [ ]:
print(get_breeze(query))

被告利國智（被告）利用不知情的代書曾廷蓉，持以不實的切結書前往潮州地政事務所，申請就上開不動產辦理繼承登記。被告的行為涉及刑法第214條的使公務員登載不實罪。

被告在104年8月30日，以所有權狀遺失為由，填寫了土地登記申請書、切結書等文件，委由代書曾廷蓉持向


In [ ]:
query = "請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境："+train_facts[2]

In [ ]:
query

'請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境：一、陳慶松及其妻左鳳竹均明知陳建妹即陳慶松之母於民國103年11月5日係將苗栗縣○○鄉○○○段000○0地號土地（下稱本案土地）贈與予陳慶松，而非買賣，竟共同基於使公務員登載不實之犯意聯絡，由陳慶松交由左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗縣苗栗地政事務所（下稱苗栗地政事務所）辦理本案土地買賣登記，使不知情之承辦公務員為形式審查後，將「登記原因：買賣」之不實事項登載於職務上所掌之土地登記謄本及地籍異動索引等公文書上，足以生損害於陳建妹及地政機關對於不動產登記之正確管理。二、案經陳建妹訴由臺灣苗栗地方檢察署檢察官偵查起訴。理由壹、證據能力部分一、被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據；被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據，刑事訴訟法第159條第1項、第159條之5第1項分別定有明文。查本案所引被告以外之人於審判外之供述，經被告陳慶松、左鳳竹及辯護人表示同意作為證據（見本院卷第77頁），本院審酌該等陳述無證明力明顯過低之情事，且依製作時之情況，尚無違法不當之瑕疵，認作為證據應屬適當，是得為證據。二、被告等所為不利於己之供述，無出於強暴、脅迫、利誘、詐欺、疲勞訊問、違法羈押或其他不正之方法，亦非違反法定障礙事由經過期間不得訊問或告知義務規定而為，依刑事訴訟法第156條第1項、第158條之2規定，應有證據能力。三、本案所引其餘非供述證據，無違反法定程序而取得之情形，依刑事訴訟法第158條之4反面規定，得為證據。貳、認定犯罪事實所憑證據及其理由訊據被告等固坦承有辦理本案土地買賣登記之事實，惟矢口否認有何使公務員登載不實之犯行，均辯稱：係被告陳慶松以新臺幣（下同）100萬元向告訴人陳建妹購買本案土地云云（見本院卷第75頁至第77頁、第208頁、第284頁）。經查：一、被告左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，至苗栗地政事務所辦理本案土地買賣登記，使不知情之承辦公務員為形式審查後，將「登記原因：買賣」之事項登載於職務上所掌之土地

In [ ]:
print(get_breeze(query))

 1. 被告陳慶松及其妻左鳳竹明知陳慶松的母親陳建妹將苗栗縣○○鄉○○○段000○0地號土地贈與予陳慶松，而非買賣，卻共同基於使公務員登載不實之犯意聯絡，由陳慶松交由左鳳竹於103年11月18日，持土地登記申請書及土地建築物改良物買賣所有權移轉契約書等文件，


In [ ]:
query = "請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境："+train_facts[3]

In [ ]:
query

'請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境：一、廖麗卿前因偽造有價證券案件，經臺灣高等法院以87年度上訴字第4352號判處有期徒刑2年，並經最高法院以88年度台上字第3995號駁回上訴確定；又因詐欺及偽造文書案件，經臺灣高等法院以88年度上訴字第3284號分別判處有期徒刑1年、3月確定；復因違反動產擔保交易法案件，經臺灣臺北地方法院以87年度店簡字第219號判處有期徒刑3月確定；再因偽造有價證券及偽造文書案件，經臺灣士林地方法院以88年度訴字第733號分別判處有期徒刑4年、8月確定；另因偽造文書案件，經本院以91年度訴字第2628號判處有期徒刑10月確定。上開案件，嗣經本院以92年度聲字第1983號裁定，定其應執行之刑為有期徒刑8年8確，再經本院以98年度聲字第4089號裁定，依中華民國九十六年罪犯減刑條例就應減刑各罪減刑後所處之刑，與不應減刑之罪所處之刑，合併定其應執行有期徒刑7年3月確定，並與其於89年間因竊盜案件，經本院以89年度易字第1253號判處有期徒刑5月，嗣經減刑為有期徒刑2月15日之案件接續執行，於民國96年4月18日縮短刑期假釋出監，假釋期間付保護管束，於96年6月27日保護管束期滿，未經撤銷，視為執行完畢。詎仍不知悔改，於97年初農曆年前，藉由介紹工作而取得鍾賢珍之國民身分證及全民健康保險卡之機會，將鍾賢珍之國民身分證及全民健康保險卡影印留存後，竟與真實姓名年籍不詳之某成年人，基於偽造私文書後進而行使及意圖為自己不法所有之詐欺取財犯意聯絡，於97年5月7日，未經鍾賢珍授權或同意，由廖麗卿在其所任職、由林合昌開設之位於臺中市○○路○段200號之電訪公司，佯以鍾賢珍之名義，申辦行動電話門號，並向該電訪公司不知情之職員偽稱：鍾賢珍為其熟識之友人，其可親自收、送件，無庸透過宅配人員配送行動電話門號空白申請書、SIM卡及附贈之手機云云，致使該公司承辦人員陷於錯誤，而將台灣大哥大股份有限公司（下稱臺灣大哥大公司）之門號0000000000號行動電話SIM卡1枚及該電訪公司附隨上開門號SIM卡所贈送之手機1支交付予廖麗卿後，另由上開真實姓名年籍不詳之成年人，擅自在台灣大哥大行動通信網路業務服務申請書之申請人簽章欄內，接續偽造「鍾賢珍」之署名共2枚，而偽造完成表彰係由「鍾賢珍」申請上開行動電話門號，性質上屬於私文書之申請書1份後，再將該

In [ ]:
print(get_breeze(query))

 1.犯罪事實：被告廖麗珍於97年5月7日，未經鍾賢珍同意，以鍾賢珍名義申辦行動電話門號0000000000號行動電話，並偽造有「鍾賢珍」之署名2枚後，持以行使於電訪公司承辦人員，而偽造有「鍾賢珍」之申請書1份，連同其持有鍾賢珍之國民身分證及全民健康


In [ ]:
query = "請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境："+train_facts[4]

In [ ]:
query#

'請依照以下犯罪情境回答我被告人做了哪些偽造或變造的行為? 犯罪情境：一、高偉為國軍退除役官兵輔導委員會臺北勞務中心（下稱北勞中心）之八德工務所主任，王士章則為上富水土保持技師事務所（下稱上富事務所）之負責人。北勞中心於民國98年5月間取得桃園市楊梅區公所「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程」（下稱老坑溪工程）之工程標，由高偉負責工程施工及工程估驗請款事宜，而老坑溪工程之監造標案，則由上富事務所得標，王士章需負責監督工程及審核廠商估驗請款文件。嗣因向北勞中心承包老坑溪主體工程之工懿營造有限公司延宕工程，經北勞中心解除契約，由擔任履約保證之慶展園藝景觀有限公司（下稱慶展公司）接續施作，慶展公司另將木作工程部分，分包予臺灣綠得電創技有限公司。後因慶展公司負責人林玨豐資金短絀欲退出工程施作，是協議由蕭翰翔出資墊付工程款，繼續以慶展公司名義負責進行工程施作，並覓得蔡盛淵、劉偉任參與工程，蔡盛淵並經北勞中心指派為工地主任，老坑溪工程之文書製作、估驗請款業務則由劉偉任負責，蕭翰翔、蔡盛淵、劉偉任、高偉、王士章就該老坑溪工程，均為從事業務之人。二、蕭翰翔明知「木棧道A」、「木棧道B」、「扶手欄杆」等工程細項，迄民國99年3月15日止，並無任何施作進度，但因負責接續施作工程之蕭翰翔資金週轉陷入困境，計畫先行請領第二期工程款以為挹注，而蔡盛淵、劉偉任、高偉受蕭翰翔請託後，雖明知「木棧道A」、「木棧道B」、「扶手欄杆」確無任何施工進度，但認上揭工程細項所需之施工材料，均已向昆晉實業股份有限公司訂貨進場，日後完成施作應屬無虞，是同意配合，而與蕭翰翔共同基於行使業務登載不實文書之犯意聯絡，由劉偉任在蕭翰翔指示下，於99年3月某日、老坑溪工程工地內，在「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日）」、「楊梅鎮老坑溪（老坑溪橋至起新橋）兩側路廊景觀工程、第二次估驗數量計算表、99/02/01～99/03/15」、「桃園縣楊梅鎮公所工程估驗計價明細表、第二次估驗（估驗期間：99/02/01～99/03/15）」等估驗請款業務文書上，虛偽登載「木棧道A」施工長度110公尺、「木棧道B」施工長度120公尺、「扶手欄杆」施工長度200公尺之不實施工進度，北勞中心派任之工地主任蔡盛淵則於「桃園縣楊梅鎮公所

In [ ]:
print(get_breeze(query))

被告蕭翰翔、蔡盛淵、劉偉任、高偉、王士章等人所犯之犯罪行為，屬於刑法第216條、第215條之行使業務上登載不實文書罪。他們在「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月12日）」、「桃園縣楊梅鎮公所公共工程施工日誌（填報日期：99年3月15日）」、


## promt 6
我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：{jedgement}

In [ ]:
query = "我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[0]

In [ ]:
query

'我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、羅智弘明知真實姓名年籍不詳之成年人劉嘉宇（音同）持有之余岳騰之國民身分證及汽車駕駛執照係來路不明之贓物（該等證件係余岳騰於民國101年1月1日18時許，在臺中市大里區仁化路沿線遺失），仍基於收受贓物之犯意，於101年7月底某時，在臺中市某處，收受余岳騰之國民身分證及汽車駕駛執照並持有之。羅智弘另與林建辰、孫育微基於行使偽造私文書之犯意聯絡，於101年8月3日某時，由羅智弘先將持有之余岳騰之國民身分證及汽車駕駛執照交付予林建辰及孫育微，並負擔租車費用，再由林建辰及孫育微於同日20時30分許，至臺中市○○區○○路0段000號1樓順大小客車租賃有限公司（下稱順大公司），由林建辰持余岳騰之國民身分證及汽車駕駛執照，冒用余岳騰名義，於中華民國小客車租賃定型化契約書暨汽車出租單（以下簡稱出租單）之承租人欄偽造「余岳騰」之簽名2枚，而偽造該契約書，表示係余岳騰租車之用意。再由孫育微在上開契約書之連帶保證人欄簽名後，共同持向該公司負責人吳慶章而行使之，並提供孫育微所有車牌號碼000-000號機車1輛供擔保，以取信於順大公司負責人吳慶章，致使吳慶章交付車牌號碼0000-00號小客車予林建辰、孫育微使用，足以生損害於余岳騰及順大公司。嗣林建辰、孫育微雖於101年8月4日22時28分許依約還車，但因於租車期間發生交通事故後逃逸，為警循線通知余岳騰本人到場說明，始查悉上情。二、案經臺中市政府警察局第六分局報告臺灣臺中地方法院檢察署檢察官偵查起訴及臺灣臺中地方法院檢察署檢察官主動檢舉偵辦並追加起訴。理由一、證據能力之審酌：(一)按「被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。」刑事訴訟法第159條第1項固定有明文。惟按「被告以外之人於審判外之陳述，雖不符前4條之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據。當事人、代理人或辯護人於法院調查證據時，知有第159條第1項不得為證據之情形，而未於言詞辯論終結前聲明異議者，視為有前項之同意。」刑事訴訟法第159條之5亦規定甚明。經查，本件證人即被告林建辰、孫育微（相對於其他被告而言）、證人余岳騰、吳慶章於警詢時所為之供述，屬被

In [ ]:
len(query)

3876

In [ ]:
print(get_breeze(query))

 1. 偽造的文書：出租單（中華民國小客車租賃定型化契約書暨汽車出租單）。
2. 偽造行為：被告羅智弘將持有的余岳騰之國民身分證及汽車駕駛執照交付給被告林建辰及孫育微，並負擔租車費用。被告林建辰及孫育微在出租單上偽造余岳騰的簽名，表示余岳騰租車之用意。
3. 損害利益於他人的地方：
a. 余岳騰：因證件被冒用，可能面臨法律責任，且其真實身份被冒用可能造成其他問題。
b. 順大公司（臺中市○○區○○路0段000號1樓順大小客車租賃有限公司）：因被告林建辰及孫育微冒用余岳騰的名義租車，順大公司可能受到損失，例如租車費用、車輛損壞等。


In [ ]:
query = "我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[1]

In [ ]:
query

'我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、賴聰質係從事經營茶葉批發買賣，緣南投縣鹿谷鄉凍頂茶葉生產合作社（下稱鹿谷茶葉合作社）於民國104年10月19日至23日舉辦鹿谷鄉104年冬季高級凍頂烏龍茶暨新品種茶葉比賽及受理報名，新品種組於同年11月14、15日繳交茶葉，烏龍種組於同年12月7、8日繳交茶葉，比賽規定須為臺灣所生產之當季茶葉方得參賽，每位社員可報名參賽1至10茶點(每茶點繳交21台斤茶葉及新臺幣【下同】2000元報名費，其中1台斤供比賽評審評鑑用)；詎賴聰質明知比賽規定須為臺灣所生產之當季茶葉方得參賽，竟意圖為自己不法之利益，基於詐欺得利之犯意，於104年7月間在越南與不知比賽規定之曾燈立洽談以在越南種植之茶葉參與前開比賽之事宜，由不知情之曾燈立負責出資向「越南龍頂茶葉責任股份有限公司」購買重量達9500台斤的越南茶葉，交給賴聰質在越南完成製茶後，自越南胡志明市進口，經我國臺中港入關後，直接載運到曾燈立之不知情小舅鍾松延開立之「玉美香」食品工廠寄放，復由賴聰質將該批茶葉運回南投縣竹山鎮租用廠房再製（全部製茶過程包括撿枝、烘焙乾燥、真空包裝的耗損），賴聰質以前開每台斤成本約360元的越南進口茶葉9030台斤(430茶點)，向鹿谷茶葉合作社分別報名參賽104年冬季新品種組5980台斤(299茶點【299×20＝5980】)及烏龍種組2620台斤(131茶點【131×20＝2620】)，並由鹿谷茶葉合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠等人身分，供賴聰質使用該等社員名義參展，賴聰質並以「自己名義」之印章蓋用「賴聰質」之印文製作烏龍組、新品種之報名表、切結書，由鹿谷茶葉合作社職員柯茂生、莊婉琪經辦（2人所涉行使偽造私文書罪嫌，均由檢察官為緩起訴處分）。嗣鹿谷茶葉合作社舉辦之104年冬季茶葉比賽結果，賴聰質以該批越南進口茶在新品種組獲得貳等獎40台斤(2茶點，其中1台斤供比賽評審評鑑用)、參等獎160台斤(8茶點)、3朵梅花獎1700台斤(85茶點)、2朵梅花獎1500台斤(75茶點)、優級獎1240台斤(62茶點)、淘汰1340台斤（67茶點）；烏龍種組獲得5朵梅花獎20台斤(1茶點)、3朵梅花獎240台斤(12茶點)、2朵梅花獎440台斤(22茶點)、

In [ ]:
len(query)

10459

In [ ]:
print(get_breeze(query))

 104年度冬季茶葉比賽新品種組104年10月14、1700台號至1700台號之茶展銷知茶生產合作社104年冬季茶葉比賽之結果，由鹿谷茶葉合作社104年10月14日舉辦鹿谷鄉104年冬季茶葉比賽暨新品種組、2200文新品種組參賽，於鹿谷茶葉合作社104年冬季茶展製作之烏龍組、新品種組參賽之104年冬季茶葉比賽新品種組參賽，於鹿谷茶葉合作社104年冬季茶展製作之烏龍組、新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比賽新品種組參賽之104年冬季茶葉比

In [ ]:
query = "我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[2]

In [ ]:
query

'我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、陳勤瀚於民國107年4、5月間某日起至107年9月29日晚上9時40分許為警查獲時止之期間內某日，可預見其所有之自用小客車（引擎號碼4G93H007322號）車前所懸掛之車號000-0000號牌照係在白色鐵板上黏貼數字與英文貼紙所製作，充作真正牌照使用之偽造牌照，猶為行使而不違背其本意，基於行使偽造特種文書之不確定犯意，駕駛上開自用小客車行駛於道路，足以生損害於公路監理機關管理汽車牌照之正確性。嗣於107年9月29日晚上9時40分許在宜蘭縣○○鎮○○路0段000號前為警查獲，並扣得上開偽造牌照1面，始悉上情。二、案經宜蘭縣政府警察局蘇澳分局報告臺灣宜蘭地方檢察署檢察官偵查起訴。理由壹、程序部分：按被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。被告以外之人於審判外之陳述，雖不符同法第159條之1至之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據；當事人、代理人或辯護人於法院調查證據時，知有第159條第1項不得為證據之情形，而未於言詞辯論終結前聲明異議者，視為有前項之同意，刑事訴訟法第159第1項、第159條之5第1項、第2項分別定有明文。本件被告及檢察官於本院準備程序、審理時就下述證據資料均不爭執證據能力，且迄至言詞辯論終結前亦未聲明異議，本院審酌各該證據資料作成時之一切客觀情況，認無違法不當之瑕疵，以之為證據應屬適當，自得為本案論罪科刑之依據，合先敘明。貳、實體部分：一、訊據被告固坦承其於107年2月28日前某日取得車牌號碼000-0000號自用小客車，且於取得時該車之車牌尚未改掛本件扣案之偽造牌照，惟否認涉犯起訴書所載之偽造車牌及行使偽造車牌犯行，辯稱：車牌不是我弄的，我不知道是誰弄的，被抓到時我才知道是偽造的。經查：(一)被告上開坦承部分，業據其於本院準備程序時所不爭執（見本院卷第30頁）。又本件員警於107年9月29日晚上9時40分許在宜蘭縣○○鎮○○路0段000號前查獲被告駕駛之車牌號碼000-0000號自用小客車前方懸掛本件扣案之偽造牌照等事實，業據被告於警詢中陳述明確（見警澳偵字第1070014371號卷第1至2頁），核與證人即本

In [ ]:
len(query)

2676

In [ ]:
print(get_breeze(query))

 判決書符合偽造文書構成要件如下：

1. 偽造的文書種類：偽造特種文書（汽車牌照）。
2. 偽造行為：被告將白色鐵板上黏貼數字與英文貼紙所製作的偽造牌照懸掛在自用小客車上，用以充作真正牌照使用。
3. 損害利益：損害於公路監理機關管理汽車牌照之正確性。

判決書內容符合以下法條：

1. 中華民國刑法第216條：行使第210條至第215條之文書者，依偽造、變造文書或登載不實事項或使登載不實事項之規定處斷。
2. 中華民國刑法第212條：偽造、變造護照、旅券、免許證、特許證及關於品行、能力服務或其他相類之證書、介紹書，足以生損害於公眾或他人者，處1年以下有期徒刑、拘役或3百元以下罰金。


In [ ]:
query = "我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[3]

In [ ]:
query

'我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、林奇穎於民國107年5月15日21時許，因涉嫌違反毒品危害防制條例案件為警查獲，在花蓮縣警察局花蓮分局豐川派出所應訊時，為免其另案遭通緝之身分為警發現及避免因施用第二級毒品案件受到刑事追訴處罰，竟基於行使偽造私文書及偽造署押之接續犯意，於同日21時55許至22時25分許止，冒用不知情之其弟「林廉智」之名義應訊，接續於附表編號1至5所示之文書及欄位內偽造「林廉智」之署名及指印（數量各如附表所示），足生損害於「林廉智」；其中附表編號5所示之文書，依其內容表示「林廉智」本人對勘察採證為同意之意思表示，並將之交還警員收執存卷而行使之，足生損害於警察機關對犯罪偵查之正確性及「林廉智」本人。嗣經檢察官傳喚林廉智到庭訊問而查悉上情。二、案經臺灣花蓮地方檢察署檢察官分案偵查起訴。理由壹、證據能力部分：按被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據；當事人、代理人或辯護人於法院調查證據時，知有第159條第1項不得為證據之情形，而未於言詞辯論終結前聲明異議者，視為有前項之同意，刑事訴訟法第159條第1項、第159條之5分別定有明文。本判決以下所引用被告以外之人於審判外之言詞或書面陳述，固屬傳聞證據，惟被告於本院審理時對於證據能力表示沒有意見，均同意作為證據等語(見本院卷第155頁)，迄至本件言詞辯論終結前，亦未聲明異議，本院審酌上開證據資料作成時之外部狀況，尚無違法取證及證明力明顯過低之瑕疵，認為以之作為證據應屬適當，依前揭刑事訴訟法第159條之5之規定，認前揭供述證據均有證據能力。貳、實體事項一、認定犯罪事實所憑之證據及理由：上揭犯罪事實，業據被告於偵查中、原審及本院審理時坦承不諱（見毒偵卷第43頁、原審卷第52頁反面、第57頁反面、本院卷第157頁），核與證人即被告之弟林廉智於偵查中之證述相符(毒偵卷第25-30頁)，並有如附表所載之文書、林廉智之簽名比對資料、內政部警政署刑事警察局107年11月8日刑生字第1078004171號鑑定書在卷

In [ ]:
len(query)

4235

In [ ]:
print(get_breeze(query))

 1.偽造署押部分
2.文書編號
3.偽造之署押種類、數量
4.附表編號1
5.署名（6枚）
6.附表編號2
7.署名（6枚）
8.附表編號3
9.署名（6枚）
10.附表編號4
11.署名（6枚）
12.附表編號5
13.署名（6枚）
14.指印（18枚）
15.合計：24枚

1.行使偽造私文書部分
2.文書編號
3.偽造之私文書
4.附表編號5
5.勘察採證同意書
6.


In [ ]:
query = "我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[4]

In [ ]:
query

'我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、邱煥煌於民國102年間某日，在其位於桃園縣平鎮市（於103年12月25日改制為桃園市平鎮區）義興街之工作室，未經警安時報有限公司（下稱警安時報）之同意，即基於偽造特種文書及偽造印文之犯意，先分別於3張警安時報空白記者證之正面貼上楊聯枝、黃仕綸（原名黃福來）、姜建旭之照片，並於空白記者證背面「單位」欄填載「新聞採訪中心」、「職稱」欄填載「記者」及「督導兼記者」、「有效期限」欄填載「自民國102年1月1日起至民國102年12月31日止」，並在「姓名」、「出生年月日」、「身分證統一編號」欄分別填載「楊聯枝」、「黃福來」、「姜建旭」及渠等3人之年籍與身分證統一編號，再於該3張記者證背面偽造「警安時報社有限公司」印文各1枚，以此方式偽造警安時報記者證3張並將上開偽造之警安時報記者證交付予楊聯枝、黃仕綸及姜建旭（楊聯枝、黃仕綸、姜建旭所涉行使偽造特種文書罪部分，業經臺灣桃園地方法院檢察署檢察官以104年度偵字第4541號為不起訴處分確定），足以生損害於警安時報對於記者證製作與管理之正確性。二、案經警安時報訴由臺灣桃園地方法院檢察署檢察官偵查後起訴。理由壹、程序事項：本案所引用之供述及非供述證據，經本院依法踐行調查證據程序，檢察官、被告邱煥煌均不爭執各該證據之證據能力，且亦查無依法應排除其證據能力之情形，是後述所引用證據之證據能力均無疑義。貳、實體事項：一、訊據被告固坦承製發上開警安時報記者證交給楊聯枝、黃仕綸、姜建旭，惟矢口否認有何偽造特種文書及偽造印文之犯意，辯稱：伊有跟警安時報採訪中心主任章國晁報告這件事情，章國晁有同意伊製發記者證給楊聯枝、黃仕綸、姜建旭云云。經查：（一）被告於102年間某日，在其工作室分別於3張警安時報空白記者證之正面貼上楊聯枝、黃仕綸、姜建旭之照片，並於空白記者證背面「單位」欄填載「新聞採訪中心」、「職稱」欄填載「記者」及「督導兼記者」、「有效期限」欄填載「自民國102年1月1日起至民國102年12月31日止」，並在「姓名」、「出生年月日」、「身分證統一編號」欄分別填載「楊聯枝」、「黃福來」、「姜建旭」及渠等3人之年籍與身分證統一編號，再於上開3張記者證背面印上「警安時報社有限公司」印文各1枚，並將上開警安時報記

In [ ]:
len(query)

3233

In [ ]:
print(get_breeze(query))

 1. 偽造的文書：警安時報記者證
2. 偽造的行為：被告未經警安時報之同意，在3張記者證正面貼上楊聯枝、黃仕綸、姜建旭的照片，並在背面填寫不實資料，如單位、職稱、有效期限等，以及偽造「警安時報社有限公司」印文。
3. 損害利益於他人的地方：
a. 偽造的記者證可能導致警安時報無法正確管理記者證，影響其業務運作。
b. 偽造的記者證可能被其他人濫用，損害警安時報的信譽和公眾對新聞報導的信任。
c. 被告的行為可能導致楊聯枝、黃仕綸、姜建旭等三人被誤認為合法記者，而獲得不正當的利益或特權。


## promt 7
我會給你一篇法律判決書內容，列出這篇判決書符合偽造文書構成要件：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：{jedgement}

In [ ]:
query = "我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[0]

In [ ]:
query

'我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、羅智弘明知真實姓名年籍不詳之成年人劉嘉宇（音同）持有之余岳騰之國民身分證及汽車駕駛執照係來路不明之贓物（該等證件係余岳騰於民國101年1月1日18時許，在臺中市大里區仁化路沿線遺失），仍基於收受贓物之犯意，於101年7月底某時，在臺中市某處，收受余岳騰之國民身分證及汽車駕駛執照並持有之。羅智弘另與林建辰、孫育微基於行使偽造私文書之犯意聯絡，於101年8月3日某時，由羅智弘先將持有之余岳騰之國民身分證及汽車駕駛執照交付予林建辰及孫育微，並負擔租車費用，再由林建辰及孫育微於同日20時30分許，至臺中市○○區○○路0段000號1樓順大小客車租賃有限公司（下稱順大公司），由林建辰持余岳騰之國民身分證及汽車駕駛執照，冒用余岳騰名義，於中華民國小客車租賃定型化契約書暨汽車出租單（以下簡稱出租單）之承租人欄偽造「余岳騰」之簽名2枚，而偽造該契約書，表示係余岳騰租車之用意。再由孫育微在上開契約書之連帶保證人欄簽名後，共同持向該公司負責人吳慶章而行使之，並提供孫育微所有車牌號碼000-000號機車1輛供擔保，以取信於順大公司負責人吳慶章，致使吳慶章交付車牌號碼0000-00號小客車予林建辰、孫育微使用，足以生損害於余岳騰及順大公司。嗣林建辰、孫育微雖於101年8月4日22時28分許依約還車，但因於租車期間發生交通事故後逃逸，為警循線通知余岳騰本人到場說明，始查悉上情。二、案經臺中市政府警察局第六分局報告臺灣臺中地方法院檢察署檢察官偵查起訴及臺灣臺中地方法院檢察署檢察官主動檢舉偵辦並追加起訴。理由一、證據能力之審酌：(一)按「被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。」刑事訴訟法第159條第1項固定有明文。惟按「被告以外之人於審判外之陳述，雖不符前4條之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據。當事人、代理人或辯護人於法院調查證據時，知有第159條第1項不得為證據之情形，而未於言詞辯論終結前聲明異議者，視為有前項之同意。」刑事訴訟法第159條之5亦規定甚明。經查，本件證人即被告林建辰、孫育微（相對於其他被告而言）、證人余岳騰、吳慶章於警詢時所為之供述，屬被告以外之人於審判外之

In [ ]:
print(get_breeze(query))

 1. 偽造的文書：出租單（小客車租賃定型化契約書暨汽車出租單）。
2. 偽造行為：被告林建辰在出租單的承租人欄偽造「余岳騰」的簽名2枚。
3. 損害利益於他人的地方：
a. 損害於被害人余岳騰，因他人冒用其姓名租車，可能導致法律責任或信用問題。
b. 損害於順大公司，因被告林建辰冒用余岳騰的名義租車，順大公司可能因此蒙受損失或受到法律責任。


In [ ]:
query = "我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[1]

In [ ]:
query

'我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、賴聰質係從事經營茶葉批發買賣，緣南投縣鹿谷鄉凍頂茶葉生產合作社（下稱鹿谷茶葉合作社）於民國104年10月19日至23日舉辦鹿谷鄉104年冬季高級凍頂烏龍茶暨新品種茶葉比賽及受理報名，新品種組於同年11月14、15日繳交茶葉，烏龍種組於同年12月7、8日繳交茶葉，比賽規定須為臺灣所生產之當季茶葉方得參賽，每位社員可報名參賽1至10茶點(每茶點繳交21台斤茶葉及新臺幣【下同】2000元報名費，其中1台斤供比賽評審評鑑用)；詎賴聰質明知比賽規定須為臺灣所生產之當季茶葉方得參賽，竟意圖為自己不法之利益，基於詐欺得利之犯意，於104年7月間在越南與不知比賽規定之曾燈立洽談以在越南種植之茶葉參與前開比賽之事宜，由不知情之曾燈立負責出資向「越南龍頂茶葉責任股份有限公司」購買重量達9500台斤的越南茶葉，交給賴聰質在越南完成製茶後，自越南胡志明市進口，經我國臺中港入關後，直接載運到曾燈立之不知情小舅鍾松延開立之「玉美香」食品工廠寄放，復由賴聰質將該批茶葉運回南投縣竹山鎮租用廠房再製（全部製茶過程包括撿枝、烘焙乾燥、真空包裝的耗損），賴聰質以前開每台斤成本約360元的越南進口茶葉9030台斤(430茶點)，向鹿谷茶葉合作社分別報名參賽104年冬季新品種組5980台斤(299茶點【299×20＝5980】)及烏龍種組2620台斤(131茶點【131×20＝2620】)，並由鹿谷茶葉合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠等人身分，供賴聰質使用該等社員名義參展，賴聰質並以「自己名義」之印章蓋用「賴聰質」之印文製作烏龍組、新品種之報名表、切結書，由鹿谷茶葉合作社職員柯茂生、莊婉琪經辦（2人所涉行使偽造私文書罪嫌，均由檢察官為緩起訴處分）。嗣鹿谷茶葉合作社舉辦之104年冬季茶葉比賽結果，賴聰質以該批越南進口茶在新品種組獲得貳等獎40台斤(2茶點，其中1台斤供比賽評審評鑑用)、參等獎160台斤(8茶點)、3朵梅花獎1700台斤(85茶點)、2朵梅花獎1500台斤(75茶點)、優級獎1240台斤(62茶點)、淘汰1340台斤（67茶點）；烏龍種組獲得5朵梅花獎20台斤(1茶點)、3朵梅花獎240台斤(12茶點)、2朵梅花獎440台斤(22茶點)、淘汰1920台斤（9

In [ ]:
print(get_breeze(query))

 104年度冬季茶葉比賽之新產品組二朵合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠等人之參賽茶品，並提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃崇合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃崇合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃崇合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃崇合、林銀騰、張鴻渠之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃崇合、林銀騰、張鴻士之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿谷茶葉合作社社員黃崇合、林銀騰、張鴻士之參賽茶品，於104年冬季茶葉比賽新產品組二朵合作社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合社提供鹿知合

In [ ]:
query = "我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[2]

In [ ]:
query

'我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、陳勤瀚於民國107年4、5月間某日起至107年9月29日晚上9時40分許為警查獲時止之期間內某日，可預見其所有之自用小客車（引擎號碼4G93H007322號）車前所懸掛之車號000-0000號牌照係在白色鐵板上黏貼數字與英文貼紙所製作，充作真正牌照使用之偽造牌照，猶為行使而不違背其本意，基於行使偽造特種文書之不確定犯意，駕駛上開自用小客車行駛於道路，足以生損害於公路監理機關管理汽車牌照之正確性。嗣於107年9月29日晚上9時40分許在宜蘭縣○○鎮○○路0段000號前為警查獲，並扣得上開偽造牌照1面，始悉上情。二、案經宜蘭縣政府警察局蘇澳分局報告臺灣宜蘭地方檢察署檢察官偵查起訴。理由壹、程序部分：按被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。被告以外之人於審判外之陳述，雖不符同法第159條之1至之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據；當事人、代理人或辯護人於法院調查證據時，知有第159條第1項不得為證據之情形，而未於言詞辯論終結前聲明異議者，視為有前項之同意，刑事訴訟法第159第1項、第159條之5第1項、第2項分別定有明文。本件被告及檢察官於本院準備程序、審理時就下述證據資料均不爭執證據能力，且迄至言詞辯論終結前亦未聲明異議，本院審酌各該證據資料作成時之一切客觀情況，認無違法不當之瑕疵，以之為證據應屬適當，自得為本案論罪科刑之依據，合先敘明。貳、實體部分：一、訊據被告固坦承其於107年2月28日前某日取得車牌號碼000-0000號自用小客車，且於取得時該車之車牌尚未改掛本件扣案之偽造牌照，惟否認涉犯起訴書所載之偽造車牌及行使偽造車牌犯行，辯稱：車牌不是我弄的，我不知道是誰弄的，被抓到時我才知道是偽造的。經查：(一)被告上開坦承部分，業據其於本院準備程序時所不爭執（見本院卷第30頁）。又本件員警於107年9月29日晚上9時40分許在宜蘭縣○○鎮○○路0段000號前查獲被告駕駛之車牌號碼000-0000號自用小客車前方懸掛本件扣案之偽造牌照等事實，業據被告於警詢中陳述明確（見警澳偵字第1070014371號卷第1至2頁），核與證人即本件查獲員警許哲銓、黃

In [ ]:
print(get_breeze(query))

 判決書內容涉及以下犯罪行為：

1. 偽造文書：被告偽造了一輛汽車的車牌（000-0000號），並將其黏貼在自己的自用小客車上使用。
2. 行使偽造特種文書：被告使用了這個偽造的車牌（特種文書），將車輛行駛於道路上。
3. 損害利益於他人：被告的行為損害了公路監理機關管理汽車牌照的正確性。

根據中華民國刑法第216條和第212條，被告被判處有期徒刑。


In [ ]:
query = "我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[3]

In [ ]:
print(get_breeze(query))

 1.偽造署押部分
2.文書編號
3.偽造之署押種類、數量
4.附表編號1
5.署名（6枚）
6.附表編號2
7.署名（6枚）
8.附表編號3
9.署名（6枚）
10.附表編號4
11.署名（6枚）
12.附表編號5
13.署名（6枚）
14.指印（18枚）
15.合計：24枚


In [ ]:
query = "我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下："+train_facts[4]

In [ ]:
query

'我會給你一篇法律判決書內容，請回答以下問題：偽造誰的什麼文書、有什麼偽造行為、損害利益於他人的地方。判決書內容如下：一、邱煥煌於民國102年間某日，在其位於桃園縣平鎮市（於103年12月25日改制為桃園市平鎮區）義興街之工作室，未經警安時報有限公司（下稱警安時報）之同意，即基於偽造特種文書及偽造印文之犯意，先分別於3張警安時報空白記者證之正面貼上楊聯枝、黃仕綸（原名黃福來）、姜建旭之照片，並於空白記者證背面「單位」欄填載「新聞採訪中心」、「職稱」欄填載「記者」及「督導兼記者」、「有效期限」欄填載「自民國102年1月1日起至民國102年12月31日止」，並在「姓名」、「出生年月日」、「身分證統一編號」欄分別填載「楊聯枝」、「黃福來」、「姜建旭」及渠等3人之年籍與身分證統一編號，再於該3張記者證背面偽造「警安時報社有限公司」印文各1枚，以此方式偽造警安時報記者證3張並將上開偽造之警安時報記者證交付予楊聯枝、黃仕綸及姜建旭（楊聯枝、黃仕綸、姜建旭所涉行使偽造特種文書罪部分，業經臺灣桃園地方法院檢察署檢察官以104年度偵字第4541號為不起訴處分確定），足以生損害於警安時報對於記者證製作與管理之正確性。二、案經警安時報訴由臺灣桃園地方法院檢察署檢察官偵查後起訴。理由壹、程序事項：本案所引用之供述及非供述證據，經本院依法踐行調查證據程序，檢察官、被告邱煥煌均不爭執各該證據之證據能力，且亦查無依法應排除其證據能力之情形，是後述所引用證據之證據能力均無疑義。貳、實體事項：一、訊據被告固坦承製發上開警安時報記者證交給楊聯枝、黃仕綸、姜建旭，惟矢口否認有何偽造特種文書及偽造印文之犯意，辯稱：伊有跟警安時報採訪中心主任章國晁報告這件事情，章國晁有同意伊製發記者證給楊聯枝、黃仕綸、姜建旭云云。經查：（一）被告於102年間某日，在其工作室分別於3張警安時報空白記者證之正面貼上楊聯枝、黃仕綸、姜建旭之照片，並於空白記者證背面「單位」欄填載「新聞採訪中心」、「職稱」欄填載「記者」及「督導兼記者」、「有效期限」欄填載「自民國102年1月1日起至民國102年12月31日止」，並在「姓名」、「出生年月日」、「身分證統一編號」欄分別填載「楊聯枝」、「黃福來」、「姜建旭」及渠等3人之年籍與身分證統一編號，再於上開3張記者證背面印上「警安時報社有限公司」印文各1枚，並將上開警安時報記者證交付予楊聯枝、黃

In [ ]:
print(get_breeze(query))

 1. 偽造了誰的文書？
在本案中，被告邱煥煌偽造了警安時報有限公司（下稱警安時報）的特種文書，即記者證。
2. 有哪些偽造行為？
被告偽造了警安時報空白記者證，在正面貼上楊聯枝、黃仕綸（原名黃福來）、姜建旭的照片，並在背面填寫不實的單位、職稱、有效期限，以及姓名、出生年月日、身分證統一編號等資訊。此外，被告還在記者證背面偽造了「警安時報社有限公司」的印文。
3. 損害利益於誰的地方？
被告的偽造行為損害了警安時報對於記者證製作與管理的正確性，以及對楊聯枝、黃仕綸、姜建旭等人的真實身份確認。


# CoT

## prompt 1


CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'

In [ ]:
query = train_facts[0]

In [ ]:
CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'

In [ ]:
len(CoT_prompt)

6502

In [ ]:
CoT_prompt

'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳

In [ ]:
print(get_breeze(CoT_prompt))

 1. 羅智弘、林建辰和孫育微涉嫌在民國101年7月底至8月4日期間，共同偽造文書並行使偽造文書。羅智弘收受來路不明的余岳騰證件，並將其交給林建辰和孫育米，共同前往順大小客車租賃有限公司租車。林建辰冒用余岳騰名義在出租單上偽造「余岳騰」簽名，孫育微在連帶保證人欄簽名，並提供機車作為擔保。他們以偽造的文件向順大公司負責人吳慶章行使，取得車輛使用權。
2. 羅智弘、林建辰和孫育微的行為涉嫌觸犯刑法第216條、第210條行使偽造私文書罪。羅智弘和林建辰的行為還涉嫌觸犯刑法第349條第1項收受贓物罪。
3. 羅智弘、林建辰和孫育微的犯罪行為均已明確，因此，羅智弘、林建辰和孫育微的犯罪行為均成立。
4. 羅智弘、林建辰和孫育米的犯罪行為具有共同犯意聯絡和行為分擔，因此，他們是共同正犯。
5. 羅智弘所犯的2罪，犯意各別，行為互殊，因此，應予分論併罰。
6. 羅智弘、林建辰和孫育米的犯罪行為均已判處有期徒刑，並均諭知易科罰金之折算標準。
7. 偽造的出租單已交付順大公司，因此，依刑法第219條規定，宣告沒收。


In [ ]:
query = train_facts[1]

In [ ]:
CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'

In [ ]:
len(CoT_prompt)

13085

In [ ]:
print(get_breeze(CoT_prompt))

之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之筆之面之筆之筆之面之筆之面之筆之面之筆之面之筆之面之筆之面之筆之面之筆之面之筆之面之筆之面之面之面之筆之面之筆之面之面之筆之面之筆之面之面之筆之面之面之面之面之面之筆之面之面之面之筆之面之面之面之筆之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面之面

In [ ]:
query = train_facts[2]

In [ ]:
CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'

In [ ]:
len(CoT_prompt)

5302

In [ ]:
print(get_breeze(CoT_prompt))

 在這個案件中，被告駕駛了一輛懸掛偽造牌照的車輛。被告於取得車輛後，發現車牌有異狀，但仍繼續使用車輛。被告於使用車輛時，已具有使用偽造牌照的不確定故意。因此，被告的行為違反了刑法第216條、第212條之行使偽造特種文書罪。在審理中，被告否認犯行，但法院根據證據，包括被告的供述、證人證詞、車輛詳細資料報表等，認定被告的犯罪事實。被告的犯罪動機、犯罪後態度、教育程度、職業等因素，被法院考慮在量刑時。最後，法院判處被告有期徒刑，並宣告易科罰金的折算標準。


In [ ]:
query = train_facts[3]

In [ ]:
CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'

In [ ]:
len(CoT_prompt)

6861

In [ ]:
print(get_breeze(CoT_prompt))

 1. 孫桂蘭與孟奇於民國83年7月10日結婚，孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之繼母。
2. 丁○○○為陳全清之妻，己○○為陳全清之弟即甲○○之兒媳，乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女。
3. 吳文錦為吳清海之長子，吳芸安則係吳清海次子吳文橂之女，為吳文橂之代位繼承人，吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女。

以上是根據原文本的內容，以繁體中文重新整理了三個案例的摘要。


In [ ]:
query = train_facts[4]

In [ ]:
CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'

In [ ]:
len(CoT_prompt)

5859

In [ ]:
print(get_breeze(CoT_prompt))

 1. 孫桂蘭與孟奇於民國83年7月10日結婚，孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡，孫桂蘭明知孟奇死亡後，其名下財產均為遺產，於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有，未經全體繼承人同意，任一繼承人不得擅自處分。孫桂蘭竟意圖為自己不法所有，基於詐欺取財及行使偽造私文書之犯意，於附表各編號所示之時間，至附表各編號所示金融機構，冒用孟奇之名義，在空白取款憑證填寫如附表各編號所示之日期、帳號及金額，並於「取款印鑑」欄，使用孟奇之印章盜蓋印文1枚，表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額，而偽造孟奇名義之取款憑證私文書後，再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之，致該金融機構承辦人員陷於錯誤，誤認孫桂蘭係經孟奇本人授權而為上開手續，如數交付如附表各編號所示金額，均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司、合作金庫商業銀行股份有限公司竹塹分行、中華郵政股份有限公司清華大學郵局、永豐商業銀行等金融機構對於孟奇帳戶資料、存款管理之正確性。


### all data under 5500（prompt長度）
情結摘要大部分還不錯，但是有時還是會出現人名

In [ ]:
CoT_prompt_ = "我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入: Text1輸出:(請用繁體中文輸出)"

In [ ]:
len(CoT_prompt_)

2694

In [ ]:
indexs = []
for i, fact in enumerate(train_facts):
  if(len(fact)<(5500-2694)):
    print(f"{i}: {len(fact)}")
    indexs.append(i)

2: 2608
17: 2766
29: 1192
42: 2585
53: 1867
78: 1525
94: 2361
106: 2013
127: 2565
188: 1259
215: 2691
245: 1777
252: 1559
253: 328
270: 2066
316: 2382
320: 280
328: 1772
335: 2475
395: 1838
398: 2755
409: 1518
420: 2499
425: 2004
426: 1350
432: 2344
438: 1930
460: 1292
490: 2315
496: 1331


In [ ]:
for index in indexs:
  query = train_facts[index]
  CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'
  print(train_facts[index])
  print(f"{index}: {len(CoT_prompt)}")
  print(get_breeze(CoT_prompt))
  print("=======================================================================================\n")

一、陳勤瀚於民國107年4、5月間某日起至107年9月29日晚上9時40分許為警查獲時止之期間內某日，可預見其所有之自用小客車（引擎號碼4G93H007322號）車前所懸掛之車號000-0000號牌照係在白色鐵板上黏貼數字與英文貼紙所製作，充作真正牌照使用之偽造牌照，猶為行使而不違背其本意，基於行使偽造特種文書之不確定犯意，駕駛上開自用小客車行駛於道路，足以生損害於公路監理機關管理汽車牌照之正確性。嗣於107年9月29日晚上9時40分許在宜蘭縣○○鎮○○路0段000號前為警查獲，並扣得上開偽造牌照1面，始悉上情。二、案經宜蘭縣政府警察局蘇澳分局報告臺灣宜蘭地方檢察署檢察官偵查起訴。理由壹、程序部分：按被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。被告以外之人於審判外之陳述，雖不符同法第159條之1至之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據；當事人、代理人或辯護人於法院調查證據時，知有第159條第1項不得為證據之情形，而未於言詞辯論終結前聲明異議者，視為有前項之同意，刑事訴訟法第159第1項、第159條之5第1項、第2項分別定有明文。本件被告及檢察官於本院準備程序、審理時就下述證據資料均不爭執證據能力，且迄至言詞辯論終結前亦未聲明異議，本院審酌各該證據資料作成時之一切客觀情況，認無違法不當之瑕疵，以之為證據應屬適當，自得為本案論罪科刑之依據，合先敘明。貳、實體部分：一、訊據被告固坦承其於107年2月28日前某日取得車牌號碼000-0000號自用小客車，且於取得時該車之車牌尚未改掛本件扣案之偽造牌照，惟否認涉犯起訴書所載之偽造車牌及行使偽造車牌犯行，辯稱：車牌不是我弄的，我不知道是誰弄的，被抓到時我才知道是偽造的。經查：(一)被告上開坦承部分，業據其於本院準備程序時所不爭執（見本院卷第30頁）。又本件員警於107年9月29日晚上9時40分許在宜蘭縣○○鎮○○路0段000號前查獲被告駕駛之車牌號碼000-0000號自用小客車前方懸掛本件扣案之偽造牌照等事實，業據被告於警詢中陳述明確（見警澳偵字第1070014371號卷第1至2頁），核與證人即本件查獲員警許哲銓、黃梓瑜於偵查中之證述相符（見107年度偵字第6518號卷第13頁背面至第14頁），並有搜索扣押筆錄、扣押物品目錄表、照片

### all data between 7000 and 5500
情節摘要能力下降，名字都過慮不掉

In [ ]:
indexs = []
for i, fact in enumerate(train_facts):
  if(len(fact)>=(5500-2694) and len(fact)<=(7000-2694)):
    print(f"{i}: {len(fact)}")
    indexs.append(i)

0: 3808
3: 4167
4: 3165
8: 3940
16: 3110
30: 4235
50: 3520
51: 3683
61: 4156
76: 3164
95: 3702
96: 3661
115: 4153
118: 3901
131: 4255
132: 4133
142: 3935
153: 3317
155: 4258
158: 4271
160: 3009
167: 4173
174: 3520
191: 3030
192: 4197
201: 3385
210: 3690
213: 2919
228: 3241
233: 3759
234: 3498
237: 3848
240: 3482
271: 3937
273: 3297
276: 2898
300: 3365
304: 4054
306: 3482
309: 3910
324: 2996
329: 3733
336: 4159
343: 4062
356: 3412
357: 3744
363: 3823
372: 3288
376: 3743
382: 4064
391: 3775
412: 4229
414: 4220
416: 2859
417: 3374
428: 3941
429: 3386
434: 4034
445: 3189
450: 3883
457: 4164
467: 4283
469: 3570
494: 3475
497: 3735


In [ ]:
for index in indexs[:10]:
  query = train_facts[index]
  CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'
  print(train_facts[index])
  print(f"{index}: {len(CoT_prompt)}")
  print(get_breeze(CoT_prompt))
  print("=======================================================================================\n")

一、羅智弘明知真實姓名年籍不詳之成年人劉嘉宇（音同）持有之余岳騰之國民身分證及汽車駕駛執照係來路不明之贓物（該等證件係余岳騰於民國101年1月1日18時許，在臺中市大里區仁化路沿線遺失），仍基於收受贓物之犯意，於101年7月底某時，在臺中市某處，收受余岳騰之國民身分證及汽車駕駛執照並持有之。羅智弘另與林建辰、孫育微基於行使偽造私文書之犯意聯絡，於101年8月3日某時，由羅智弘先將持有之余岳騰之國民身分證及汽車駕駛執照交付予林建辰及孫育微，並負擔租車費用，再由林建辰及孫育微於同日20時30分許，至臺中市○○區○○路0段000號1樓順大小客車租賃有限公司（下稱順大公司），由林建辰持余岳騰之國民身分證及汽車駕駛執照，冒用余岳騰名義，於中華民國小客車租賃定型化契約書暨汽車出租單（以下簡稱出租單）之承租人欄偽造「余岳騰」之簽名2枚，而偽造該契約書，表示係余岳騰租車之用意。再由孫育微在上開契約書之連帶保證人欄簽名後，共同持向該公司負責人吳慶章而行使之，並提供孫育微所有車牌號碼000-000號機車1輛供擔保，以取信於順大公司負責人吳慶章，致使吳慶章交付車牌號碼0000-00號小客車予林建辰、孫育微使用，足以生損害於余岳騰及順大公司。嗣林建辰、孫育微雖於101年8月4日22時28分許依約還車，但因於租車期間發生交通事故後逃逸，為警循線通知余岳騰本人到場說明，始查悉上情。二、案經臺中市政府警察局第六分局報告臺灣臺中地方法院檢察署檢察官偵查起訴及臺灣臺中地方法院檢察署檢察官主動檢舉偵辦並追加起訴。理由一、證據能力之審酌：(一)按「被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。」刑事訴訟法第159條第1項固定有明文。惟按「被告以外之人於審判外之陳述，雖不符前4條之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據。當事人、代理人或辯護人於法院調查證據時，知有第159條第1項不得為證據之情形，而未於言詞辯論終結前聲明異議者，視為有前項之同意。」刑事訴訟法第159條之5亦規定甚明。經查，本件證人即被告林建辰、孫育微（相對於其他被告而言）、證人余岳騰、吳慶章於警詢時所為之供述，屬被告以外之人於審判外之言詞陳述，惟被告3人及檢察官於本院調查證據時，知有上情而未於言詞辯論終結前聲明異議，本院審酌該等陳述或係執法機關依法定

### all data > 7000
100000字以上出現無意義重複句子

In [ ]:
indexs = []
for i, fact in enumerate(train_facts):
  if(len(fact)>=(7000-2694)):
    print(f"{i}: {len(fact)}")
    indexs.append(i)

1: 10391
5: 15733
6: 18254
7: 5803
9: 12680
10: 5408
11: 4795
12: 5852
13: 10314
14: 5051
15: 10470
18: 12726
19: 18319
20: 4403
21: 6425
22: 16352
23: 7365
24: 9444
25: 7972
26: 12839
27: 13858
28: 9095
31: 5008
32: 14802
33: 4947
34: 6546
35: 7600
36: 5192
37: 8613
38: 6990
39: 10345
40: 4580
41: 6459
43: 7911
44: 4453
45: 6302
46: 12222
47: 5653
48: 5514
49: 17804
52: 5582
54: 10270
55: 6568
56: 8368
57: 17285
58: 5118
59: 10358
60: 14021
62: 17514
63: 12845
64: 9240
65: 15791
66: 8275
67: 7409
68: 9262
69: 8952
70: 9449
71: 5040
72: 13912
73: 6615
74: 9583
75: 6647
77: 9006
79: 19252
80: 5794
81: 7425
82: 14691
83: 5415
84: 4996
85: 7281
86: 5313
87: 6468
88: 7405
89: 11931
90: 8225
91: 6072
92: 13164
93: 7042
97: 8005
98: 5127
99: 11374
100: 12576
101: 6141
102: 4817
103: 10861
104: 6220
105: 10050
107: 5524
108: 7237
109: 8183
110: 5596
111: 12313
112: 11428
113: 9339
114: 18998
116: 5237
117: 11363
119: 11884
120: 6465
121: 9516
122: 6339
123: 8478
124: 9336
125: 9481
126: 9071


In [ ]:
for index in indexs[:10]:
  query = train_facts[index]
  CoT_prompt = f'我會給你三個範例 Example 1, Example 2 and Example 3 (輸入:Text,輸出:Context Summary) Example1 : Text輸入:孫桂蘭與孟奇於民國83年7月10日結婚,孫桂蘭係孟奇之子女即孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉之繼母。孟奇於96年1月22日死亡,孫桂蘭明知孟奇死亡後,其名下財產均為遺產,於遺產分割前屬孫桂蘭、孟慶明、孟慶虹、孟慶經、孟慶緯、孟蓉全體繼承人公同共有,未經全體繼承人同意,任一繼承人不得擅自處分,竟意圖為自己不法所有,基於詐欺取財及行使偽造私文書之犯意,於附表各編號所示之時間,至附表各編號所示金融機構,冒用孟奇之名義,在空白取款憑證填寫如附表各編號所示之日期、帳號及金額,並於「取款印鑑」欄,使用孟奇之印章盜蓋印文1枚,表示係孟奇欲自附表各編號所示金融機構帳戶內提領附表各編號所示提款金額,而偽造孟奇名義之取款憑證私文書後,再連同上開帳戶存摺持之向不知情之附表各編號所示金融機構承辦人員辦理取款而行使之,致該金融機構承辦人員陷於錯誤,誤認孫桂蘭係經孟奇本人授權而為上開手續,如數交付如附表各編號所示金額,均足以生損害於孟慶明、孟慶虹、孟慶經、孟慶緯(已歿)、孟蓉之權益及日盛國際商業銀行股份有限公司《下稱日盛商業銀行》、合作金庫商業銀行股份有限公司竹塹分行《下稱合作金庫商業銀行竹塹分行》、中華郵政股份有限公司清華大學郵局《下稱清華大學郵局》、永豐商業銀行股份有限公司《下稱永豐商業銀行》等金融機構對於孟奇帳戶資料、存款管理之正確性。 Context Summary輸出:在這個案件中,涉及到一段婚姻關係,並有繼承人之間的財產分割。一名已故的個人的遺產成為了糾紛的焦點。在其去世後,其配偶及子女成為了繼承人。然而,配偶明知其去世,卻企圖以詐欺和偽造私文書的方式來獲取他們的共同財產。她冒用已故人的名義,填寫了取款憑證並偽造了他的印章,使金融機構誤以為是合法的取款要求。最終,金融機構執行了這些虛偽的指示,導致繼承人之間的權益受損,也對涉及的金融機構造成了損害。这些行為涉及偽造私文書、詐欺取財,並對繼承人和金融機構產生了不良影響。 Example 2 : Text輸入:丁○○○為陳全清之妻,己○○為陳全清之弟即甲○○之兒媳,乙○○、陳全清、甲○○、廖陳專、丙○○、陳素美、陳月娥(下稱乙○○等7人)均為被繼承人陳秋火之子女,緣陳秋火於民國107年1月27日死亡,乙○○等7人均為陳秋火之法定繼承人。丁○○○、己○○明知陳秋火業已死亡,無法再以其名義為任何法律行為,且陳秋火所遺留之存款由全體繼承人繼承,應以全體繼承人名義具領款項,或全體繼承人同意委任代理人,依據繼承之程序,始得提領款項,丁○○○、己○○竟未得乙○○之同意,共同基於行使偽造私文書之犯意聯絡,將其等保管之陳秋火之存摺、印章取出,於107年1月29日9時55分許,共同前往位於苗栗縣○○鎮○○路0號之玉山商業銀行股份有限公司(下稱玉山銀行)後龍分行,冒用陳秋火之名義,申辦:將陳秋火之玉山銀行後龍分行定期存款(帳號依序分別為000000000號、000000000號、000000000號,金額均為新臺幣(下同)10萬元,共計30萬元)解約,存入陳秋火之同銀行帳號為0000000000000號帳戶(下稱甲帳戶)後,甲帳戶餘額為102萬2325元,再自甲帳戶提領現金25萬元,並將餘額77萬2325元匯至不知情之陳永聰(為甲○○之子、己○○之夫,另經臺灣苗栗地方檢察署檢察官為不起訴處分)之同銀行帳號0000000000000號帳戶(下稱乙帳戶)等交易,復持上開存摺、印章交由不知情之玉山銀行後龍分行承辦人員盜蓋於內部傳票、取款憑條上(偽造之文件名稱、交易摘要及金額、欄位、盜蓋印章之印文及數量均詳如附表所載),以利用該承辦人員偽造屬私文書之上開內部傳票、取款憑條後,據以交由該承辦人員辦理而行使之,致使承辦之玉山銀行人員誤認陳秋火尚在世並授權,而將上開陳秋火之定期存款解約及存入甲帳戶內之存款102萬2325萬元憑摺支取,除其中25萬元提領現金外,餘77萬2325元匯入乙帳戶,足以生損害於陳秋火之繼承人及玉山銀行後龍分行對於存款帳戶管理之正確性。Context Summary輸出:這宗案件涉及家族關係與遺產問題。已故個人的遺產引發了爭議。他的配偶與子女是合法繼承人,但配偶卻企圖透過詐欺手段來侵占共同財產。她冒用已故人的身份,偽造文件並盗用印章,讓金融機構誤以為是合法的交易。最終,金融機構執行了這些虛偽指示,導致繼承人的權益受損,也給金融機構造成損失。這些行為牽涉到偽造文件和詐騙,對繼承人和金融機構都帶來不良影響。Example 3: Text輸入:吳文錦係吳清海(民國108年2月2日死亡)之長子,吳芸安則係吳清海次子吳文樑(於103年3月22日死亡)之女,為吳文樑之代位繼承人,吳淑妮、吳金桂、吳淑惠、吳月嬌亦為吳清海之子女,惟吳文錦於吳清海過世後,為辦理吳清海之後事相關事宜時,明知不得再以吳清海之名義提領存款,須經全體繼承人同意或授權,始得提領存款,竟未經全體繼承人之同意或授權,基於行使偽造私文書之犯意,接續於108年2月11日、108年2月20日,持其所保管吳清海之雲林縣○○鄉○○0○號:000-0000000000000號,下稱四湖鄉農會帳戶)之存摺、印鑑,前往雲林縣○○鄉○○村○○○路00號之雲林縣四湖鄉農會,冒用吳清海之名義,在該農會之取款憑條存戶簽章欄上,填寫領取新臺幣(下同)337,600元、7,290元,並偽簽「吳清海」之署押1枚,盜蓋「吳清海」印文2枚(詳如附表所示),而偽造吳清海同意提領337,600元、7,290元之私文書,並持該偽造之取款憑條交予上開農會之不知情承辦人員而行使之,使吳文錦自前開帳戶提領共344,890元,足生損害於吳清海全體繼承人及四湖鄉農會對客戶存款管理之正確性。Context Summary輸出:這宗案件涉及家族關係和遺產問題。已故個人的子女是合法繼承人,但其中一人明知不得未經全體繼承人同意或授權就提領存款。他使用偽造文件,包括存摺和印鑑,前往當地農會冒充已故父親的身份,提領款項,造成損害。这涉及偽造文件和可能的詐騙行為,對繼承人和農會的存款管理產生不良影響。 請你模仿Example 1 , Example 2, Example 3,來生成以下一個輸入(Text1)之context摘要,禁止出現人名。Text1 輸入:{query} Text1輸出:(請用繁體中文輸出)'
  print(train_facts[index])
  print(f"{index}: {len(CoT_prompt)}")
  print(get_breeze(CoT_prompt))
  print("=======================================================================================\n")

一、賴聰質係從事經營茶葉批發買賣，緣南投縣鹿谷鄉凍頂茶葉生產合作社（下稱鹿谷茶葉合作社）於民國104年10月19日至23日舉辦鹿谷鄉104年冬季高級凍頂烏龍茶暨新品種茶葉比賽及受理報名，新品種組於同年11月14、15日繳交茶葉，烏龍種組於同年12月7、8日繳交茶葉，比賽規定須為臺灣所生產之當季茶葉方得參賽，每位社員可報名參賽1至10茶點(每茶點繳交21台斤茶葉及新臺幣【下同】2000元報名費，其中1台斤供比賽評審評鑑用)；詎賴聰質明知比賽規定須為臺灣所生產之當季茶葉方得參賽，竟意圖為自己不法之利益，基於詐欺得利之犯意，於104年7月間在越南與不知比賽規定之曾燈立洽談以在越南種植之茶葉參與前開比賽之事宜，由不知情之曾燈立負責出資向「越南龍頂茶葉責任股份有限公司」購買重量達9500台斤的越南茶葉，交給賴聰質在越南完成製茶後，自越南胡志明市進口，經我國臺中港入關後，直接載運到曾燈立之不知情小舅鍾松延開立之「玉美香」食品工廠寄放，復由賴聰質將該批茶葉運回南投縣竹山鎮租用廠房再製（全部製茶過程包括撿枝、烘焙乾燥、真空包裝的耗損），賴聰質以前開每台斤成本約360元的越南進口茶葉9030台斤(430茶點)，向鹿谷茶葉合作社分別報名參賽104年冬季新品種組5980台斤(299茶點【299×20＝5980】)及烏龍種組2620台斤(131茶點【131×20＝2620】)，並由鹿谷茶葉合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠等人身分，供賴聰質使用該等社員名義參展，賴聰質並以「自己名義」之印章蓋用「賴聰質」之印文製作烏龍組、新品種之報名表、切結書，由鹿谷茶葉合作社職員柯茂生、莊婉琪經辦（2人所涉行使偽造私文書罪嫌，均由檢察官為緩起訴處分）。嗣鹿谷茶葉合作社舉辦之104年冬季茶葉比賽結果，賴聰質以該批越南進口茶在新品種組獲得貳等獎40台斤(2茶點，其中1台斤供比賽評審評鑑用)、參等獎160台斤(8茶點)、3朵梅花獎1700台斤(85茶點)、2朵梅花獎1500台斤(75茶點)、優級獎1240台斤(62茶點)、淘汰1340台斤（67茶點）；烏龍種組獲得5朵梅花獎20台斤(1茶點)、3朵梅花獎240台斤(12茶點)、2朵梅花獎440台斤(22茶點)、淘汰1920台斤（96茶點），復由該合作社將每1台斤茶葉依得獎品級密封包裝成標示產地為臺灣之茶葉罐禮盒，交還給賴聰質販售或由鹿谷茶葉合作社

# summary

## prompt 1

summary_prompt = ' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points.'

In [ ]:
summary_prompt = f' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points: {train_facts[0]}'

In [ ]:
summary_prompt

' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points: 一、羅智弘明知真實姓名年籍不詳之成年人劉嘉宇（音同）持有之余岳騰之國民身分證及汽車駕駛執照係來路不明之贓物（該等證件係余岳騰於民國101年1月1日18時許，在臺中市大里區仁化路沿線遺失），仍基於收受贓物之犯意，於101年7月底某時，在臺中市某處，收受余岳騰之國民身分證及汽車駕駛執照並持有之。羅智弘另與林建辰、孫育微基於行使偽造私文書之犯意聯絡，於101年8月3日某時，由羅智弘先將持有之余岳騰之國民身分證及汽車駕駛執照交付予林建辰及孫育微，並負擔租車費用，再由林建辰及孫育微於同日20時30分許，至臺中市○○區○○路0段000號1樓順大小客車租賃有限公司（下稱順大公司），由林建辰持余岳騰之國民身分證及汽車駕駛執照，冒用余岳騰名義，於中華民國小客車租賃定型化契約書暨汽車出租單（以下簡稱出租單）之承租人欄偽造「余岳騰」之簽名2枚，而偽造該契約書，表示係余岳騰租車之用意。再由孫育微在上開契約書之連帶保證人欄簽名後，共同持向該公司負責人吳慶章而行使之，並提供孫育微所有車牌號碼000-000號機車1輛供擔保，以取信於順大公司負責人吳慶章，致使吳慶章交付車牌號碼0000-00號小客車予林建辰、孫育微使用，足以生損害於余岳騰及順大公司。嗣林建辰、孫育微雖於101年8月4日22時28分許依約還車，但因於租車期間發生交通事故後逃逸，為警循線通知余岳騰本人到場說明，始查悉上情。二、案經臺中市政府警察局第六分局報告臺灣臺中地方法院檢察署檢察官偵查起訴及臺灣臺中地方法院檢察署檢察官主動檢舉偵辦並追加起訴。理由一、證據能力之審酌：(一)按「被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。」刑事訴訟法第159條第1項固定有明文。惟按「被告以外之人於審判外之陳述，雖不符前4條之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據。當事人、代理人或辯護人於法院調查證據時，知有第159條第1項不得為證據之情形，而未於言詞辯論終結前聲明異議者，

In [ ]:
summary = get_breeze(summary_prompt)

In [ ]:
summary

' \n\n以下是一個描述羅智弘、林建辰和孫育微的法律案件的故事。\n\n羅智弘、林建辰和孫育微是朋友，他們一起在臺中市生活。有一天，羅智弘從一個陌生的人「劉嘉宇」（音同）那裡收到了余岳騰的國民身分證和汽車駕駛執照，但他們不知道這些證件是如何得到的。羅智弘決定使用這些證件來租車，所以他交給林建辰和孫育微。\n\n林建辰和孫育微前往順大小客車租賃有限公司（順大公司），林建辰使用了余岳騰的證件，在出租單上偽造余岳騰的簽名，表示余岳騰要租車。孫育微在保證人欄上簽名，表示他願意為租車承擔責任。然後，他們向順大公司的負責人吳慶章出示了證件，並提供了孫育微的機車作為擔保。吳慶章相信他們，所以把車交給了他們。\n\n然而，在租車期間，林建辰和孫育微發生了交通事故，然後逃逸。後來，警察找到了余岳騰，他才知道自己的證件被盜用。警察調查後，發現了羅智弘、林建辰和孫育微的犯罪行為。\n\n在法庭上，羅智弘、林建辰和孫育微承認了他們的罪行。法官認為，羅智弘和林建辰、孫育微共同偽造文書，造成了損害給余岳騰和順大公司。因此，法官判處了他們不同的刑罰，包括罰金和有期徒刑。'

In [ ]:
len(summary)

467

In [ ]:
summary_prompt = f' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points: {train_facts[1]}'

In [ ]:
summary_prompt

' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points: 一、賴聰質係從事經營茶葉批發買賣，緣南投縣鹿谷鄉凍頂茶葉生產合作社（下稱鹿谷茶葉合作社）於民國104年10月19日至23日舉辦鹿谷鄉104年冬季高級凍頂烏龍茶暨新品種茶葉比賽及受理報名，新品種組於同年11月14、15日繳交茶葉，烏龍種組於同年12月7、8日繳交茶葉，比賽規定須為臺灣所生產之當季茶葉方得參賽，每位社員可報名參賽1至10茶點(每茶點繳交21台斤茶葉及新臺幣【下同】2000元報名費，其中1台斤供比賽評審評鑑用)；詎賴聰質明知比賽規定須為臺灣所生產之當季茶葉方得參賽，竟意圖為自己不法之利益，基於詐欺得利之犯意，於104年7月間在越南與不知比賽規定之曾燈立洽談以在越南種植之茶葉參與前開比賽之事宜，由不知情之曾燈立負責出資向「越南龍頂茶葉責任股份有限公司」購買重量達9500台斤的越南茶葉，交給賴聰質在越南完成製茶後，自越南胡志明市進口，經我國臺中港入關後，直接載運到曾燈立之不知情小舅鍾松延開立之「玉美香」食品工廠寄放，復由賴聰質將該批茶葉運回南投縣竹山鎮租用廠房再製（全部製茶過程包括撿枝、烘焙乾燥、真空包裝的耗損），賴聰質以前開每台斤成本約360元的越南進口茶葉9030台斤(430茶點)，向鹿谷茶葉合作社分別報名參賽104年冬季新品種組5980台斤(299茶點【299×20＝5980】)及烏龍種組2620台斤(131茶點【131×20＝2620】)，並由鹿谷茶葉合作社提供鹿谷茶葉合作社社員黃洙合、林銀騰、張鴻渠等人身分，供賴聰質使用該等社員名義參展，賴聰質並以「自己名義」之印章蓋用「賴聰質」之印文製作烏龍組、新品種之報名表、切結書，由鹿谷茶葉合作社職員柯茂生、莊婉琪經辦（2人所涉行使偽造私文書罪嫌，均由檢察官為緩起訴處分）。嗣鹿谷茶葉合作社舉辦之104年冬季茶葉比賽結果，賴聰質以該批越南進口茶在新品種組獲得貳等獎40台斤(2茶點，其中1台斤供比賽評審評鑑用)、參等獎160台斤(8茶點)、3朵梅花獎1700台斤(85茶點)、2朵梅花獎1500台斤(75茶點)、優級獎1240台

In [ ]:
len(summary_prompt)

10537

In [ ]:
summary = get_breeze(summary_prompt)

In [ ]:
summary

' 104年度冬季茶葉合作社舉辦之104年冬季茶葉比賽新產品組參賽茶葉比賽之新產品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶品組參賽茶葉（430茶點）之茶

In [ ]:
summary_prompt = f' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points: {train_facts[2]}'

In [ ]:
len(summary_prompt)

2754

In [ ]:
summary = get_breeze(summary_prompt)

In [ ]:
summary

' 陳勤瀚在2018年4、5月間，發現他所擁有的自用小客車（引擎號碼4G93H007322號）上的車牌有異狀，但依然繼續使用該車。他於2018年9月29日被警方查獲，發現該車前所懸掛的車牌是偽造的，以白色鐵板上黏貼數字與英文貼紙所製作。陳勤瀚被以行使偽造特種文書罪起訴，並被判處有期徒刑。'

In [ ]:
summary_prompt = f' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points: {train_facts[3]}'

In [ ]:
len(summary_prompt)

4313

In [ ]:
summary = get_breeze(summary_prompt)

In [ ]:
summary

' 1.犯罪事實：\n\n林奇穎於民國107年5月15日21時許，涉嫌違反毒品危害防制條例案件為警查獲，在花蓮縣警察局花蓮分局豐川派出所應訊時，為免其另案遭通緝之身分為警發現及避免因施用第二級毒品案件受到刑事追訴處罰，竟基於行使偽造私文書及偽造署押之接續犯意，於同日21時55許至22時25分許止，冒用不知情之其弟「林廉智」之名義應訊，接續於附表編號1至5所示之文書及欄位內偽造「林廉智」之署名及指印（數量各如附表所示），足生損害於「林廉智」；其中附表編號5所示之文書，依其內容表示「林廉智」本人對勘察採證為同意之意思表示，並將之交還警員收執存卷而行使之，足生損害於警察機關對犯罪偵查之正確性及「林廉智」本人。\n\n2.證據能力：\n\n被告以外之人於審判外之言詞或書面陳述，除法律有規定者外，不得作為證據。被告以外之人於審判外之陳述，雖不符刑事訴訟法第159條之1至第159條之4之規定，而經當事人於審判程序同意作為證據，法院審酌該言詞陳述或書面陳述作成時之情況，認為適當者，亦得為證據。\n\n3.實體事項：\n\n(一)認定犯罪事實所憑之證據及理由：\n\n上揭犯罪事實，業據被告於偵查中、原審及本院審理時坦承不諱，核與證人即被告之弟林廉智於偵查中之證述相符，並有如附表所載之文書、林廉智之簽名比對資料、內政部警政署刑事警察局107年11月8日刑生字第1078004171號鑑定書在卷可佐，足見被告之任意性自白與事實相符，應可採信。\n\n(二)論罪：\n\n1.按刑法第217條第1項所稱之「偽造署押」，係指行為人單純冒用本人名義在文件上簽名或為民法第3條第3項所稱指印之類似簽名之行為者而言（最高法院80年度台非字第277號判決意旨參照）。若在制式之書類上偽造他人署名，已為一定意思表示，具有申請書或收據等性質者，則係犯偽造文書罪（最高法院85年度台非字第146號判決意旨參照）。準此，刑法上偽造文書罪章所指之文書，除必須具備有體性、文字性、持續性、意思性等要件外，尚必須存有足以表彰一定之製作名義人之名義性時，該等記載方可認為係刑法上之文書，否則若僅為表彰某人簽名意思之署名或簽押，並不具有表彰一定製作名義人之名義性存在，該等記載即無從認為係刑法上之文書，而只能認為係單純之署押。是倘行為人係以簽名之意，於文件上簽名，且該簽名僅在於表示簽名者個人身份，以作為人格同一性之證明，除此之外，

In [ ]:
len(summary)

1354

In [ ]:
summary_prompt = f' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points: {train_facts[4]}'

In [ ]:
len(summary_prompt)

3311

In [ ]:
summary = get_breeze(summary_prompt)

In [ ]:
summary

' \n\n邱煥煌在民國102年未經警安時報公司的同意，擅自在3張空白的記者證上，貼上楊聯枝、黃仕綸和姜建旭的照片，並填寫了他們個人的資料，包括姓名、出生年月日和身分證統一編號。他接著在記者證背面偽造了「警安時報社有限公司」的印文，然後將這些偽造的記者證交給楊聯枝、黃仕綸和姜建旭。由於邱煥煌的行為違反了法律，他被以偽造特種文書和偽造印文罪名起訴。\n\n在審理過程中，邱煥煌否認犯行，聲稱他已經獲得了警安時報採訪中心主任章國晁的同意。然而，章國晁和其他證人表示，他們沒有授權邱煥煌自行核發記者證，且申請記者證的程序需要經過總社審核。法院最後認定，邱煥煌的辯詞是臨訟卸責之詞，無足採信。\n\n法院考量邱煥煌的犯罪動機、犯罪情節和手段，以及他犯後的態度，最後判處他有期徒刑六個月，得易科罰金。此外，法院也宣告沒收偽造的「警安時報社有限公司」印文3枚。'

In [ ]:
summary_prompt = f' I will give you a description of the facts of a legal judgment, and help me summarize it into a scenario about 500 words without listing points: {train_facts[5]}'

In [ ]:
len(summary_prompt)

15879

In [ ]:
summary = get_breeze(summary_prompt)

In [ ]:
summary

'知認、知認之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之之�0之�之之�之�之之之之之�之之之之之之�074之之之之之�04 之��之之之之之之之�之之之��之之之��之�之�之�之�之�之�之�之之之之之�之�之之、之之之之之之之之之之之之之�之�之�之�之之�之�之�之�之之之�之�之�之�之�之�之�之�之之之之�之�之�之�之�之�之�之�之之之之�之�之�之�之�之�之�之拾之�之之之�之�之�之�之�之之�之五之之3�之之�之�之�之之�之�之之之�4�之�之�之�之�之之之�之之之之�之77之之2之�之�之�之�之之之之之之之之之之之之之之之之之之之之之之之之之之之之1之之之之之之之之五之之之之之1之  1之之之之之之之之之15 之 之之之之之之之之之之之之之之之之之之之之之之之�之之之之之之之之�之之之之之之之之之之之之之五之�之之之之之之之之之之二之五之之之之之之之之之之4之五之之5之51之之之五之五之之之之二之二之二之二之二之五之之之即之5之之之之之之之之之之之之之之之之之之之之之之之之之之之共二之2之1之合之之之之之之之之之之之之之之共之1之1之1之即之1之1之1之之之51之1之之五�之五������之五�之之之之五�之五�之五即之五即五�之二之二�之五�五�之五之之五��之之之之之之之之瑞��之五�之之五��之之之五�之五� 之五�之之之之1之五�之之1之1之五�之五�二�2之2之2之二到二之之之二之之之之之之之之之之之之之之五�五即�之1之五即�二�之二夫之之之之之五之之之之之之之之之之之之之之之之之之之之之二之之之之之之之之之之之之之五之二之之二之二之二之之之之五之之之即之之之1之1之二之二施二合二五之五�就二施釋之即性之五�即��即之五�即性之5之1之1之五之即五性之即之五之施二之1之五之1之五即五�之五度之五之之五�之五�即五�之五�即性之五之五經即性之五五即五合二之二之之五即之五之之五之五五即�即五�之五形�之之之之之之之之之之五之 於之之之 之二之之五之 合性之五即旨 之之之之之即性之即合理之之五

In [ ]:
sum = 0
for i, data in enumerate(train_facts):
  if(len(data)>10000):
    sum = sum+1
    print(f"{i}: {len(data)}")


1: 10391
5: 15733
6: 18254
9: 12680
13: 10314
15: 10470
18: 12726
19: 18319
22: 16352
26: 12839
27: 13858
32: 14802
39: 10345
46: 12222
49: 17804
54: 10270
57: 17285
59: 10358
60: 14021
62: 17514
63: 12845
65: 15791
72: 13912
79: 19252
82: 14691
89: 11931
92: 13164
99: 11374
100: 12576
103: 10861
105: 10050
111: 12313
112: 11428
114: 18998
117: 11363
119: 11884
133: 10843
134: 19704
136: 10867
138: 13632
139: 11178
140: 13178
148: 10258
149: 16309
150: 11582
156: 16796
159: 10115
161: 13075
162: 12159
163: 15644
172: 12966
176: 11597
182: 10470
184: 14403
185: 14770
189: 15128
190: 19277
193: 16301
196: 13932
209: 10393
218: 13588
225: 14889
226: 13701
230: 15854
241: 16867
244: 12739
251: 13777
255: 10172
256: 18243
261: 11158
267: 10855
269: 10841
275: 13943
277: 15482
278: 18998
279: 10491
280: 13669
281: 13845
284: 10295
286: 10301
287: 12556
289: 12458
296: 15361
297: 12857
299: 10046
305: 10699
315: 18318
321: 11938
323: 11154
338: 11896
362: 15011
364: 14459
366: 11178
367: 1178

In [ ]:
sum

124